
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.6fd905b0fb6f.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.9c7b200444b8.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.9730e47fc7c0.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Drug Design by Pharmacophore and Virtual Screening Approach - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322CDC468A3891B500002F9284D4C56A.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="1.7.0.post5+fa37e81" />

<meta name="ncbi_domain" content="pharmaceuticals" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC9145410/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9145410/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Pharmaceuticals" /><meta name="citation_title" content="Drug Design by Pharmacophore and Virtual Screening Approach" /><meta name="citation_author" content="Deborah Giordano" /><meta name="citation_author_institution" content="National Research Council, Institute of Food Science, Via Roma 64, 83110 Avellino, Italy; ti.rnc.asi@onadroig.harobed (D.G.); moc.liamg@oinegra.ainotna.airam (M.A.A.)" /><meta name="citation_author" content="Carmen Biancaniello" /><meta name="citation_author_institution" content="Doctorate School in Computational and Quantitative Biology, University of Naples &#x0201c;Federico II&#x0201d;, 80100 Naples, Italy; moc.liamg@39lnbnemrac" /><meta name="citation_author" content="Maria Antonia Argenio" /><meta name="citation_author_institution" content="National Research Council, Institute of Food Science, Via Roma 64, 83110 Avellino, Italy; ti.rnc.asi@onadroig.harobed (D.G.); moc.liamg@oinegra.ainotna.airam (M.A.A.)" /><meta name="citation_author" content="Angelo Facchiano" /><meta name="citation_author_institution" content="National Research Council, Institute of Food Science, Via Roma 64, 83110 Avellino, Italy; ti.rnc.asi@onadroig.harobed (D.G.); moc.liamg@oinegra.ainotna.airam (M.A.A.)" /><meta name="citation_publication_date" content="2022/05" /><meta name="citation_issue" content="5" /><meta name="citation_volume" content="15" /><meta name="citation_doi" content="10.3390/ph15050646" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC9145410/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC9145410/" /><meta name="citation_pmid" content="35631472" /><meta name="DC.Title" content="Drug Design by Pharmacophore and Virtual Screening Approach" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Multidisciplinary Digital Publishing Institute  (MDPI)" /><meta name="DC.Contributor" content="Deborah Giordano" /><meta name="DC.Contributor" content="Carmen Biancaniello" /><meta name="DC.Contributor" content="Maria Antonia Argenio" /><meta name="DC.Contributor" content="Angelo Facchiano" /><meta name="DC.Date" content="2022 May" /><meta name="DC.Identifier" content="10.3390/ph15050646" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Drug Design by Pharmacophore and Virtual Screening Approach" /><meta property="og:type" content="article" /><meta property="og:description" content="Computer-aided drug discovery techniques reduce the time and the costs needed to develop novel drugs. Their relevance becomes more and more evident with the needs due to health emergencies as well as to the diffusion of personalized medicine. Pharmacophore ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9145410/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC9145410/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Pharmaceuticals%20(Basel)%22[journal]" class="navlink">Pharmaceuticals (Basel)</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC9145410
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC9145410/pdf/pharmaceuticals-15-00646.pdf" class="int-view">PDF (6.7M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/9145410/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/9145410/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="9145410" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9145410%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="qVros8rPmAnuObK8oOQF30sTpWnf9Wzx9QQkggToB806vQooIQq2BSUX4mzN0d5y">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9145410%2F&amp;text=Drug%20Design%20by%20Pharmacophore%20and%20Virtual%20Screening%20Approach" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9145410%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9145410/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC9145410/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/35631472/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC9145410/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/35631472/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC9145410/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/9145410/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Pharmaceuticals%20(Basel)%22[journal]" class="navlink">Pharmaceuticals (Basel)</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC9145410
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-pharmaceuticals.png" alt="Logo of pharmaceuticals" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.mdpi.com/journal/pharmaceuticals" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Pharmaceuticals (Basel).</span> 2022 May; 15(5): 646. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2022 May 23. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.3390%2Fph15050646" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.3390/ph15050646</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC9145410</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/35631472">35631472</a></div></div></div><h1 class="content-title">Drug Design by Pharmacophore and Virtual Screening Approach</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Giordano%20D%5BAuthor%5D" class="affpopup" co-rid="_co_idm140392623062160" co-class="co-affbox">Deborah Giordano</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Biancaniello%20C%5BAuthor%5D" class="affpopup" co-rid="_co_idm140392664510976" co-class="co-affbox">Carmen Biancaniello</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Argenio%20MA%5BAuthor%5D" class="affpopup" co-rid="_co_idm140392625116480" co-class="co-affbox">Maria Antonia Argenio</a>,<sup>1</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Facchiano%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140392625114608" co-class="co-affbox">Angelo Facchiano</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140392623062160"><h3 class="no_margin">Deborah Giordano</h3><p><sup>1</sup>National Research Council, Institute of Food Science, Via Roma 64, 83110 Avellino, Italy; <a href="mailto:dev@null" data-email="ti.rnc.asi@onadroig.harobed" class="oemail">ti.rnc.asi@onadroig.harobed</a> (D.G.); <a href="mailto:dev@null" data-email="moc.liamg@oinegra.ainotna.airam" class="oemail">moc.liamg@oinegra.ainotna.airam</a> (M.A.A.)</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Giordano%20D%5BAuthor%5D">Deborah Giordano</a></div></div><div id="_co_idm140392664510976"><h3 class="no_margin">Carmen Biancaniello</h3><p><sup>2</sup>Doctorate School in Computational and Quantitative Biology, University of Naples &#x0201c;Federico II&#x0201d;, 80100 Naples, Italy; <a href="mailto:dev@null" data-email="moc.liamg@39lnbnemrac" class="oemail">moc.liamg@39lnbnemrac</a></p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Biancaniello%20C%5BAuthor%5D">Carmen Biancaniello</a></div></div><div id="_co_idm140392625116480"><h3 class="no_margin">Maria Antonia Argenio</h3><p><sup>1</sup>National Research Council, Institute of Food Science, Via Roma 64, 83110 Avellino, Italy; <a href="mailto:dev@null" data-email="ti.rnc.asi@onadroig.harobed" class="oemail">ti.rnc.asi@onadroig.harobed</a> (D.G.); <a href="mailto:dev@null" data-email="moc.liamg@oinegra.ainotna.airam" class="oemail">moc.liamg@oinegra.ainotna.airam</a> (M.A.A.)</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Argenio%20MA%5BAuthor%5D">Maria Antonia Argenio</a></div></div><div id="_co_idm140392625114608"><h3 class="no_margin">Angelo Facchiano</h3><p><sup>1</sup>National Research Council, Institute of Food Science, Via Roma 64, 83110 Avellino, Italy; <a href="mailto:dev@null" data-email="ti.rnc.asi@onadroig.harobed" class="oemail">ti.rnc.asi@onadroig.harobed</a> (D.G.); <a href="mailto:dev@null" data-email="moc.liamg@oinegra.ainotna.airam" class="oemail">moc.liamg@oinegra.ainotna.airam</a> (M.A.A.)</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Facchiano%20A%5BAuthor%5D">Angelo Facchiano</a></div></div></div></div><div class="contrib-group half_rhythm fm-editor">Osvaldo Andrade Santos-Filho, <span class="fm-role">Academic Editor</span></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140392665443952_ai idm140392623062288_ai idm140392624334320_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140392665443952_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140392665443952_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140392665443952_ai" style="display:none"><div class="fm-affl" id="af1-pharmaceuticals-15-00646"><sup>1</sup>National Research Council, Institute of Food Science, Via Roma 64, 83110 Avellino, Italy; <a href="mailto:dev@null" data-email="ti.rnc.asi@onadroig.harobed" class="oemail">ti.rnc.asi@onadroig.harobed</a> (D.G.); <a href="mailto:dev@null" data-email="moc.liamg@oinegra.ainotna.airam" class="oemail">moc.liamg@oinegra.ainotna.airam</a> (M.A.A.)</div><div class="fm-affl" id="af2-pharmaceuticals-15-00646"><sup>2</sup>Doctorate School in Computational and Quantitative Biology, University of Naples &#x0201c;Federico II&#x0201d;, 80100 Naples, Italy; <a href="mailto:dev@null" data-email="moc.liamg@39lnbnemrac" class="oemail">moc.liamg@39lnbnemrac</a></div><div id="c1-pharmaceuticals-15-00646"><sup>*</sup>Correspondence: <a href="mailto:dev@null" data-email="ti.rnc.asi@onaihccaf.olegna" class="oemail">ti.rnc.asi@onaihccaf.olegna</a></div></div><div class="fm-article-notes hide half_rhythm" id="idm140392665443952_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2022 Apr 30; Accepted 2022 May 21.</div></div><div class="permissions half_rhythm hide" id="idm140392665443952_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2022 by the authors.</div><div class="license half_rhythm">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>).</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-length="99" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p>No new data were created or analyzed in this study. Data sharing is not applicable to this article.</p></dd></dl></div><div id="abstract-a.g.b.q" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.g.b.qtitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Computer-aided drug discovery techniques reduce the time and the costs needed to develop novel drugs. Their relevance becomes more and more evident with the needs due to health emergencies as well as to the diffusion of personalized medicine. Pharmacophore approaches represent one of the most interesting tools developed, by defining the molecular functional features needed for the binding of a molecule to a given receptor, and then directing the virtual screening of large collections of compounds for the selection of optimal candidates. Computational tools to create the pharmacophore model and to perform virtual screening are available and generated successful studies. This article describes the procedure of pharmacophore modelling followed by virtual screening, the most used software, possible limitations of the approach, and some applications reported in the literature.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">structure-based pharmacophore modeling, ligand-based pharmacophore modeling, virtual screening, drug discovery, bioinformatics, computational biology</span></div></div><div id="sec1-pharmaceuticals-15-00646" class="tsec sec"><h2 class="head no_bottom_margin" id="sec1-pharmaceuticals-15-00646title">1. Introduction</h2><p class="p p-first">Computer-Aided Drug Discovery (CADD) investigates molecular properties to develop novel therapeutic solutions by way of computational tools and data resources. In its broadest meaning, it includes computational approaches for designing or selecting compounds as potential candidates before they are synthesized and tested for their biological activity [<a href="#B1-pharmaceuticals-15-00646" rid="B1-pharmaceuticals-15-00646" class=" bibr popnode">1</a>]. Bioinformatics and computational tools offer an in silico approach to reducing costs and times, i.e., the factors that influence the progress of the research and, in the specific field of drug development, limit the possibilities of fighting more pathologies. To date, in vitro screening is expensive and time-consuming, and alternatives are highly desirable. Virtual Screening (VS) is a CADD method that involves in silico screening of a library of chemical compounds, to identify those that are most likely to bind to a specific target [<a href="#B2-pharmaceuticals-15-00646" rid="B2-pharmaceuticals-15-00646" class=" bibr popnode">2</a>]. In this way, it is possible to reduce the impact of these limiting factors on drug discovery, meeting the needs due to health emergencies, as well as the spread of personalized medicine. This process can be speeded up using pharmacophore models used as the query with which compound libraries can be searched to pull out molecules of interest with desired properties. Indeed, pharmacophore-based methods are widely used tools in CADD and are of great interest in the chemo-informatics field [<a href="#B2-pharmaceuticals-15-00646" rid="B2-pharmaceuticals-15-00646" class=" bibr popnode">2</a>], since they find many applications in drug discovery projects including not only the virtual screening but also scaffold hopping, lead optimization, ligand profiling, target identification, multi-target drug or de novo drug design.</p><p>The concept of a pharmacophore was coined in the 19th century when Langley first suggested that certain drug molecules might act on particular receptors. Only later, with the discovery of Salvarsan by Paul Elrich, the selectivity of drug&#x02013;target interactions was recognized. Several years of experimentation confirmed its therapeutic effect. This discovery had found support in the statement of Emil Fisher who, following his research, had defined the concept "Lock &#x00026; Key" in 1894, according to which a ligand and its receptor fits like a key with its lock to interact with the on top of each other through a chemical bond [<a href="#B3-pharmaceuticals-15-00646" rid="B3-pharmaceuticals-15-00646" class=" bibr popnode">3</a>]. </p><p>Hence, historically the term &#x0201c;pharmacophore&#x0201d; was used to indicate the functional or structural capacity of a compound with specific characteristics towards a biological target [<a href="#B4-pharmaceuticals-15-00646" rid="B4-pharmaceuticals-15-00646" class=" bibr popnode">4</a>]. </p><p>Schueler provided the basis for our modern understanding of a pharmacophore, defined by the International Union of Pure and Applied Chemistry (IUPAC) as "the ensemble of steric and electronic features that is necessary to ensure the optimal supra-molecular interactions with a specific biological target structure and to trigger (or to block) its biological response" [<a href="#B3-pharmaceuticals-15-00646" rid="B3-pharmaceuticals-15-00646" class=" bibr popnode">3</a>,<a href="#B5-pharmaceuticals-15-00646" rid="B5-pharmaceuticals-15-00646" class=" bibr popnode">5</a>,<a href="#B6-pharmaceuticals-15-00646" rid="B6-pharmaceuticals-15-00646" class=" bibr popnode">6</a>].</p><p>Pharmacophoric modelling is based on the theory that having common chemical functionalities, and maintaining a similar spatial arrangement, leads to biological activity on the same target. The chemical characteristics of a molecule capable of creating interactions with its ligand are represented in the pharmacophoric model as geometric entities such as spheres, planes and vectors. The most important pharmacophoric feature types are: hydrogen bond acceptors (HBAs); hydrogen bond donors (HBDs); hydrophobic areas (H); positively and negatively ionizable groups (PI/NI); aromatic groups (AR); and metal coordinating areas (<a href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f001/" target="figure" class="fig-table-link figpopup" rid-figpopup="pharmaceuticals-15-00646-f001" rid-ob="ob-pharmaceuticals-15-00646-f001" co-legend-rid="lgnd_pharmaceuticals-15-00646-f001"><span>Figure 1</span></a>). Additional size restrictions in the form of a shape or exclusion volumes (XVOL)&#x02014;forbidden areas&#x02014;can be added to represent the size and the shape of the binding pocket [<a href="#B7-pharmaceuticals-15-00646" rid="B7-pharmaceuticals-15-00646" class=" bibr popnode">7</a>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pharmaceuticals-15-00646-f001" co-legend-rid="lgnd_pharmaceuticals-15-00646-f001"><a href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f001/" target="figure" rid-figpopup="pharmaceuticals-15-00646-f001" rid-ob="ob-pharmaceuticals-15-00646-f001"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140392626979072"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9145410_pharmaceuticals-15-00646-g001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pharmaceuticals-15-00646-g001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9145410/bin/pharmaceuticals-15-00646-g001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140392626979072"><a target="object" rel="noopener" href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f001/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pharmaceuticals-15-00646-f001"><div><a class="figpopup" href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f001/" target="figure" rid-figpopup="pharmaceuticals-15-00646-f001" rid-ob="ob-pharmaceuticals-15-00646-f001">Figure 1</a></div><!--caption a7--><div class="caption"><p>Pharmacophoric features. Main pharmacophoric feature types are represented by geometric entities and include: 1&#x02014;hydrogen bond acceptor (HBA), 2&#x02014;hydrogen bond donor (HBD), 3&#x02014;negative ionizable (NI), 4&#x02014;positive ionizable (PI), 5&#x02014;hydrophobic (H), 6&#x02014;aromatic (AR), 7&#x02014;exclusion volume (XVOL).</p></div></div></div><p>Since the models themselves do not focus on actual atoms, but on chemical functionalities, they are good tools in recognizing similarities between molecules. Pharmacophore activity is independent of the scaffold, and this explains why similar biological events can be triggered by chemically divergent molecules. The use of these models as a query allows performing searches in the large libraries of compounds made available on the computational platform in order to select molecules of interest for the next vs. or in the chemo-informatics field [<a href="#B7-pharmaceuticals-15-00646" rid="B7-pharmaceuticals-15-00646" class=" bibr popnode">7</a>]. </p><p class="p p-last">Pharmacophore models can be generated using two different approaches depending on the input data employed for model construction, which are, namely, &#x0201c;structure-based&#x0201d; and &#x0201c;ligand-based&#x0201d; pharmacophore modelling. The structure-based approach uses the structural information of the target proteins like enzymes or receptors, to identify compounds that can potentially be used as a drug. On the other hand, the ligand-based approach consists of the development of 3D pharmacophore models and modelling quantitative structure-activity relationship (QSAR) or quantitative structure-property relationship (QSPR), using only the physicochemical properties of known ligand molecules for drug development. The choice of the best approach to use depends on several factors such as data availability, data quality, computational resources and also the intended use of the generated pharmacophore models. In the following paragraphs, we describe both strategies and their implementation in virtual screening is provided below to guide the non-experts on this topic in their application by explaining the basic concepts these methods are based on.</p></div><div id="sec2-pharmaceuticals-15-00646" class="tsec sec"><h2 class="head no_bottom_margin" id="sec2-pharmaceuticals-15-00646title">2. Structure-Based Pharmacophore Modelling</h2><p class="p p-first">The structure-based approach owns its name to the fact that the three-dimensional structure of a macromolecule target is the essential prerequisite to obtaining a structure-based pharmacophore. The 3D structure of a protein provides significant details at the atomic level that can be very useful for the design or discovery of new drugs. As previously outlined, a pharmacophore is an abstract picture showing the stereo-electronic features, which make a ligand bioactive toward a specific target, and this type of information can be extracted from the 3D structure of the protein target in its holo or apo form. Typically, the workflow of the structure-based approach (<a href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pharmaceuticals-15-00646-f002" rid-ob="ob-pharmaceuticals-15-00646-f002" co-legend-rid="lgnd_pharmaceuticals-15-00646-f002"><span>Figure 2</span></a>, left side of the flow) consists of different steps: protein preparation, identification or prediction of ligand binding site, pharmacophore features generation, and the selection of relevant features for ligand activity [<a href="#B8-pharmaceuticals-15-00646" rid="B8-pharmaceuticals-15-00646" class=" bibr popnode">8</a>,<a href="#B9-pharmaceuticals-15-00646" rid="B9-pharmaceuticals-15-00646" class=" bibr popnode">9</a>]. </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pharmaceuticals-15-00646-f002" co-legend-rid="lgnd_pharmaceuticals-15-00646-f002"><a href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f002/" target="figure" rid-figpopup="pharmaceuticals-15-00646-f002" rid-ob="ob-pharmaceuticals-15-00646-f002"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140392626975584"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9145410_pharmaceuticals-15-00646-g002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pharmaceuticals-15-00646-g002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9145410/bin/pharmaceuticals-15-00646-g002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140392626975584"><a target="object" rel="noopener" href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f002/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pharmaceuticals-15-00646-f002"><div><a class="figpopup" href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f002/" target="figure" rid-figpopup="pharmaceuticals-15-00646-f002" rid-ob="ob-pharmaceuticals-15-00646-f002">Figure 2</a></div><!--caption a7--><div class="caption"><p>Pharmacophore modeling workflow. A pharmacophore model can be generated using two approaches. In the structure-based approach (on the left side), the structural information of the macromolecular target, alone or bounded to a ligand, is evaluated for its correctness and used to analyze the ligand binding sites in order to derive and select ligand complementary characteristics composing the pharmacophore models. In the ligand-based approach (on the right side), the first step is the construction of a training set made up of known active ligands structures. From each of them, conformers are generated and then used to extrapolate common features that are combined in different pharmacophore models. The resultant models undergo a validation and refinement procedure to obtain the final models exploitable for practical applications.</p></div></div></div><p>The 3D structure of the target or the ligand&#x02013;target complex is the required starting point of this methodology. The RCSB Protein Data Bank (PDB) [<a href="http://www.rcsb.org">www.rcsb.org</a>] (accessed on 20 May 2022) includes thousands of protein structures at high resolution alone or in the presence of a bound ligand, mainly solved by X-ray crystallography or NMR spectroscopy. In case experimental structural data are lacking, computational techniques such as homology modelling [<a href="#B10-pharmaceuticals-15-00646" rid="B10-pharmaceuticals-15-00646" class=" bibr popnode">10</a>] may be an alternative strategy to retrieve a 3D model and machine learning-based methods are successfully applied to retrieve protein structures, a powerful example is ALPHAFOLD2 [<a href="#B11-pharmaceuticals-15-00646" rid="B11-pharmaceuticals-15-00646" class=" bibr popnode">11</a>]. Molecular docking is another method to study in silico the interaction of a known active compound towards a specific receptor and derive protein-ligand complexes from the most favorable binding poses [<a href="#B12-pharmaceuticals-15-00646" rid="B12-pharmaceuticals-15-00646" class=" bibr popnode">12</a>].</p><p>A practical tip for those who want to use this approach is to perform a deep analysis of the quality of input data before using the target structure since it directly influences the quality of the pharmacophore model. For this reason, the protein structure preparation is the first step to carry out by evaluating, for instance, the residues&#x02019; protonation states, the position of hydrogen atoms (that are absent in X-ray solved structures), the presence of non-protein groups which may have or not some functional roles, eventual missing residues or atoms, the stereochemical and energetic parameters accounting for the general quality and biological-chemical sense of the investigated target, also in the case of experimentally solved structures that may contain errors [<a href="#B8-pharmaceuticals-15-00646" rid="B8-pharmaceuticals-15-00646" class=" bibr popnode">8</a>]. </p><p>Once the target structure is identified and critically evaluated, ligand-binding site detection is the next crucial step. This can be manually inferred by analyzing the area including residues suggested to have a key role from experimental data such as site-directed mutagenesis or X-ray structures of the protein co-crystallized with ligands, but it also requires time and expert knowledge of the target and ligands. Nowadays this purpose can be easily and quickly achieved using bioinformatics tools based on different methods which inspect the protein surface to search for potential ligand-binding sites according to various properties (evolutionary, geometric, energetic, statistical, or a combination of them), and which are particularly useful if the ligand information is not available [<a href="#B13-pharmaceuticals-15-00646" rid="B13-pharmaceuticals-15-00646" class=" bibr popnode">13</a>]. Examples of computer programs developed for this purpose are GRID [<a href="#B14-pharmaceuticals-15-00646" rid="B14-pharmaceuticals-15-00646" class=" bibr popnode">14</a>] and LUDI [<a href="#B15-pharmaceuticals-15-00646" rid="B15-pharmaceuticals-15-00646" class=" bibr popnode">15</a>]. The first one, as suggested by the name, is a grid-based method that uses different molecules or functional groups to sample a specific protein region defined with a regular grid to identify grid points with which they make energetically favorable interactions, thus generating molecular interaction fields [<a href="#B14-pharmaceuticals-15-00646" rid="B14-pharmaceuticals-15-00646" class=" bibr popnode">14</a>]. The second one predicts potential interaction sites using the knowledge from distributions of non-bonded contacts in experimental structures or geometric rules [<a href="#B15-pharmaceuticals-15-00646" rid="B15-pharmaceuticals-15-00646" class=" bibr popnode">15</a>]. </p><p>Since the goal of the structure-based method is to generate a pharmacophore model from the interactions between an active molecule (the ligand) and its protein target (enzyme or receptor), the characterization of the ligand-binding site can be used to derive a map of interaction and to build accordingly one or more pharmacophore hypotheses describing the type and the spatial arrangement of the chemical features required for a ligand to interact with the residues of the binding region. Initially, many features are detected with this approach and therefore only those that are essential for ligand bioactivity should be selected and incorporated into the final model to have a more reliable and selective pharmacophore hypothesis [<a href="#B9-pharmaceuticals-15-00646" rid="B9-pharmaceuticals-15-00646" class=" bibr popnode">9</a>,<a href="#B16-pharmaceuticals-15-00646" rid="B16-pharmaceuticals-15-00646" class=" bibr popnode">16</a>]. This operation can be accomplished in different ways, such as removing features that do not strongly contribute to the energy binding, identifying the most conserved interaction if multiple protein&#x02013;ligands structures exist, preserving residue with key functions from sequence alignments or variation analysis, and incorporating spatial constraints from the receptor information [<a href="#B8-pharmaceuticals-15-00646" rid="B8-pharmaceuticals-15-00646" class=" bibr popnode">8</a>].</p><p class="p p-last">If the structure of a protein&#x02013;ligand complex is available, the pharmacophore features generation and selection can be achieved more accurately. The 3D information of the ligand in its bioactive conformation directs the identification and the spatial disposition of the pharmacophore features in correspondence with its functional groups directly involved in the interactions with the target. Moreover, the presence of the receptor allows accounting for spatial restrictions from the binding site shape through the addition of the exclusion volumes. In this ideal situation, pharmacophore models of high quality can be obtained. In case the available structural data do not include a bound ligand, the pharmacophore modelling depends only on the target structure which can be analyzed to detect all possible ligands interaction points in the binding sites and, therefore, to compute the complementary features that a ligand should match to potentially bind the receptor. However, in the absence of a ligand counterpart, several pharmacophoric features are calculated and approximately positioned resulting in less accurate models that should be manually refined [<a href="#B3-pharmaceuticals-15-00646" rid="B3-pharmaceuticals-15-00646" class=" bibr popnode">3</a>,<a href="#B16-pharmaceuticals-15-00646" rid="B16-pharmaceuticals-15-00646" class=" bibr popnode">16</a>,<a href="#B17-pharmaceuticals-15-00646" rid="B17-pharmaceuticals-15-00646" class=" bibr popnode">17</a>].</p></div><div id="sec3-pharmaceuticals-15-00646" class="tsec sec"><h2 class="head no_bottom_margin" id="sec3-pharmaceuticals-15-00646title">3. Ligand-Based Pharmacophore Modelling</h2><p class="p p-first">So far, we have mentioned the importance of the availability of the structure for modelling pharmacophores suitable to accommodate the best interaction features. However, even if this still represents one of the best strategies for modelling a pharmacophore, often the target molecule is not available. To overcome this problem, an alternative approach to structural-based modelling is represented by ligand-based pharmacophore modelling.</p><p>The starting point of this strategy is the knowledge of active compounds, which bind the same protein target with a similar orientation [<a href="#B4-pharmaceuticals-15-00646" rid="B4-pharmaceuticals-15-00646" class=" bibr popnode">4</a>] that could allow the extraction of chemical features in common and, therefore, the pharmacophore construction. Due to the unknown bioactive conformations of the input compounds, a critical step, before the extraction of the shared features, is the generation of ligand conformers, so that, from each set of conformers, at list one should correspond to the bioactive conformation of the ligand [<a href="#B3-pharmaceuticals-15-00646" rid="B3-pharmaceuticals-15-00646" class=" bibr popnode">3</a>].</p><p>In more detail, two datasets are necessary to generate a ligand-based pharmacophore, i.e., a training set and a test set. The training set composition differs according to the algorithm employed and the data availability, varying from the simplest, composed of at least two active compounds, to the most complex, for which it is possible to set multiple compounds with a different activity. The test set, instead, should contain as many active and inactive structurally different compounds as possible, preferring the inclusion of experimentally confirmed inactive compounds and compounds judged as active by experimentally proven direct interaction and with suitable activity cutoffs (i.e., low EC50/IC50 and high binding affinity values). In the case of unknown inactive ligands, decoy molecules could be used [<a href="#B6-pharmaceuticals-15-00646" rid="B6-pharmaceuticals-15-00646" class=" bibr popnode">6</a>], which are available from a specific repository such as DUD (Directory of Useful Decoys) [<a href="#B18-pharmaceuticals-15-00646" rid="B18-pharmaceuticals-15-00646" class=" bibr popnode">18</a>] or by generating them by service as DUD-E [<a href="#B19-pharmaceuticals-15-00646" rid="B19-pharmaceuticals-15-00646" class=" bibr popnode">19</a>]. In addition to what could be extrapolated from pertinent literature, the source of molecules to be employed in the construction of the dataset could be chemical databases such as OpenPHACTS [<a href="#B20-pharmaceuticals-15-00646" rid="B20-pharmaceuticals-15-00646" class=" bibr popnode">20</a>], ChEMBL [<a href="#B21-pharmaceuticals-15-00646" rid="B21-pharmaceuticals-15-00646" class=" bibr popnode">21</a>] or Drugbank [<a href="#B22-pharmaceuticals-15-00646" rid="B22-pharmaceuticals-15-00646" class=" bibr popnode">22</a>], for compounds with a target-based activity, ChEMBL, PubChem Bioassay [<a href="#B23-pharmaceuticals-15-00646" rid="B23-pharmaceuticals-15-00646" class=" bibr popnode">23</a>] and Tox21 [<a href="#B24-pharmaceuticals-15-00646" rid="B24-pharmaceuticals-15-00646" class=" bibr popnode">24</a>] for research on the inactive molecules. In principle, all the algorithms appointed to the ligand-based pharmacophore modelling, starting from the active compounds of the training set, extrapolate from them common chemical functionalities, related for instance to the presence of hydrogen bond donors or acceptors, of a positively or negatively charged ionizable group, of aromatic ring systems or hydrophobic areas, and use these features for model design. This step is followed by an improvement of the pharmacophore model including shape features and spatial constrains which take care of specific regions occupied by inactive molecules [<a href="#B25-pharmaceuticals-15-00646" rid="B25-pharmaceuticals-15-00646" class=" bibr popnode">25</a>]. The pharmacophore models obtained are ranked by the best ability to fit the active molecules but not the inactive ones. The best models obtained from this first phase are chosen for the validation and refinement using the test set. The purpose of this second phase is to evaluate if the generated models can fit most of the active molecules rejecting the inactive ones, to select only models which show the major sensitivity and specificity. A summary of the described process is described in <a href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f002/" target="figure" class="fig-table-link figpopup" rid-figpopup="pharmaceuticals-15-00646-f002" rid-ob="ob-pharmaceuticals-15-00646-f002" co-legend-rid="lgnd_pharmaceuticals-15-00646-f002"><span>Figure 2</span></a> (right side of the flow).</p><p>Despite the accuracy of the pharmacophore construction being highly dependent on the quality of the training set, its final composition is conditioned by the algorithm used for modelling [<a href="#B7-pharmaceuticals-15-00646" rid="B7-pharmaceuticals-15-00646" class=" bibr popnode">7</a>]. In general, ligand-based pharmacophore modelling could be performed following two main strategies: the common feature pharmacophore and the QSAR based pharmacophore.</p><p>In the common feature approach, generally, an algorithm of alignment is involved, its purpose is basically to extrapolate and align specific and common features: starting from a defined feature for each active element the algorithm combines it with the corresponding features of the other molecules, from this alignment a novel feature, combination of the previous, is generated; reiterating this procedure for each set of overlapping features a pharmacophore of shared feature is built. This is for example the case of the espresso algorithm implemented in LigandScout [<a href="#B26-pharmaceuticals-15-00646" rid="B26-pharmaceuticals-15-00646" class=" bibr popnode">26</a>,<a href="#B27-pharmaceuticals-15-00646" rid="B27-pharmaceuticals-15-00646" class=" bibr popnode">27</a>], which achieved the pharmacophore modelling by dividing the procedure previously described into six steps exploiting various algorithms and two alignment methods: the first, based on a combination of shared pharmacophore feature and the second, based on a merged pharmacophore feature. Finally, the common features present in all training compounds are included in the common features models and all features of the aligned compounds are summated in a merged feature pharmacophore [<a href="#B28-pharmaceuticals-15-00646" rid="B28-pharmaceuticals-15-00646" class=" bibr popnode">28</a>]. Another example of similar functionality is given by the HipHop algorithm provided by DiscoveryStudio [<a href="#B29-pharmaceuticals-15-00646" rid="B29-pharmaceuticals-15-00646" class=" bibr popnode">29</a>], which builds several pharmacophore models starting from identifying the common chemical features arrangement of the training set structures. Beginning with a few groups of features, which are extended until no more shared configuration features are found, several models are obtained and ranked based on the fitness of the training compounds. Models are refined by the HipHop Refinement algorithm, which adds space restrains imposed by the structures of inactive compounds.</p><p>QSAR (Quantitative structure-activity relationships) based pharmacophore is a mathematical model that tries to find statistical correlations between structures and functions, quantifying the impacts in a biological activity of specific structural modifications in existing or predicted molecules [<a href="#B30-pharmaceuticals-15-00646" rid="B30-pharmaceuticals-15-00646" class=" bibr popnode">30</a>]. Based on the idea that molecules with similar physico-chemical properties show a similar binding affinity for a protein target, nowadays the QSAR approach is used in drug discovery to build statistical models derived from these correlations exploiting for the prediction of the bioavailability, the ADMET profile (toxicity), adsorption, distribution, metabolism, binding affinity, biological activity and elimination of novel compounds [<a href="#B31-pharmaceuticals-15-00646" rid="B31-pharmaceuticals-15-00646" class=" bibr popnode">31</a>].</p><p>There are six classifications of the QSAR approaches, from the 1D to the 6D-QSAR, according to the different dimensions of the method (<a href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="pharmaceuticals-15-00646-f003" rid-ob="ob-pharmaceuticals-15-00646-f003" co-legend-rid="lgnd_pharmaceuticals-15-00646-f003"><span>Figure 3</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pharmaceuticals-15-00646-f003" co-legend-rid="lgnd_pharmaceuticals-15-00646-f003"><a href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f003/" target="figure" rid-figpopup="pharmaceuticals-15-00646-f003" rid-ob="ob-pharmaceuticals-15-00646-f003"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140392626972768"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9145410_pharmaceuticals-15-00646-g003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pharmaceuticals-15-00646-g003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9145410/bin/pharmaceuticals-15-00646-g003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140392626972768"><a target="object" rel="noopener" href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f003/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pharmaceuticals-15-00646-f003"><div><a class="figpopup" href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f003/" target="figure" rid-figpopup="pharmaceuticals-15-00646-f003" rid-ob="ob-pharmaceuticals-15-00646-f003">Figure 3</a></div><!--caption a7--><div class="caption"><p>QSAR classifications. The QSAR approaches are classified from the 1D to the 6D-QSAR according to the different dimensions of the descriptors implemented in the method.</p></div></div></div><p class="p">1D-QSAR takes into account a single physico-chemical property of the ligand, for an example the pKa value. In the 2D-QSAR, instead, affinity is correlated with structural patterns, while in the 3D-QSAR with the 3D structure of the ligand and its interactions. The concept extends as dimension rises: 4D-QSAR incorporates an ensemble of ligand configurations in 3D-QSAR, 5D-QSAR adds to 4D-QSAR various induced-fit models, 6D-QSAR implements 5D-QSAR with different solvation models [<a href="#B25-pharmaceuticals-15-00646" rid="B25-pharmaceuticals-15-00646" class=" bibr popnode">25</a>].</p><div id="sec-a.h.c.j" class="sec sec-last"><h3 id="sec-a.h.c.jtitle">Pharmacophore Models Validation</h3><p class="p p-first">Once one or more pharmacophore models have been computed, a validation step is crucial before their implementation for practical purposes. Pharmacophore validation could be performed by exploiting several methods, such as the goodness of hit list (GH), receiver operating characteristic (ROC) curves construction, Fischer&#x02019;s method, or other statistical analysis, which relies on screening a test set and decoy set (if needed) to evaluate the model ability to distinguish active and inactive molecules and provide an estimation of its quality [<a href="#B9-pharmaceuticals-15-00646" rid="B9-pharmaceuticals-15-00646" class=" bibr popnode">9</a>]. </p><p>Mainly, the quality of a model can be described by four parameters: the sensitivity (capacity to detect active compounds), the specificity (capacity to exclude the inactive molecules), the yield of actives (the ratio between true positives and the number of hits) and the enrichment factor (which relates the yield of actives to the composition of the screening dataset) [<a href="#B7-pharmaceuticals-15-00646" rid="B7-pharmaceuticals-15-00646" class=" bibr popnode">7</a>].</p><p>The GH scoring method consists of calculating the percentage of sensitivity, the percentage of the yield of actives the enrichment factor and the G&#x000fc;ner&#x02013;Henry score, which gives an evaluation of the efficiency of the screening dataset search and can vary from 0 to 1 where 1 is the value for the ideal model [<a href="#B32-pharmaceuticals-15-00646" rid="B32-pharmaceuticals-15-00646" class=" bibr popnode">32</a>]. The ROC curve shows the enrichment power of a model plotting the sensitivity against 1&#x02014;specificity (the false positive rate). The area under this curve (AUC) gives a measure of the pharmacophore&#x02019;s performance and it is useful for multiple models evaluation. AUC can vary from 0 to 1, where 1 corresponds to an ideal case in which all the active compounds are detected at first, 0 to the collection of the inactive ones at first and 0.5 to random results [<a href="#B33-pharmaceuticals-15-00646" rid="B33-pharmaceuticals-15-00646" class=" bibr popnode">33</a>]. Fisher&#x02019;s randomization test validation method is instead used to analyze the significance of the statistical correlation between structure and biological activity [<a href="#B34-pharmaceuticals-15-00646" rid="B34-pharmaceuticals-15-00646" class=" bibr popnode">34</a>]. Regression analysis can also be applied to check for the correlation between the compounds predicted as active and those whose bioactivity is experimentally confirmed.</p><p class="p p-last">In case the validation process reveals low-quality results, manual refinement could be a way to improve its performance by applying some changes in chemical features&#x02019; selection or definition settings, training or test sets composition or in the pharmacophore building procedure, but a new model generation with a different approach and algorithm should not be excluded.</p></div></div><div id="sec4-pharmaceuticals-15-00646" class="tsec sec"><h2 class="head no_bottom_margin" id="sec4-pharmaceuticals-15-00646title">4. Pharmacophore-Based Virtual Screening</h2><p class="p p-first">Among the strategies for the identification of new bioactive substances, VS techniques play a prominent role. VS tools are important in drug discovery as they increase the speed of the bioactive molecule discovery process through computational simulations by selecting from large libraries the compounds that are most likely to interact with the identified target. In addition, VS identifies compounds that may be toxic or have unfavorable pharmacodynamic (for example, potency, affinity, selectivity) and pharmacokinetic (for example, absorption, metabolism, bioavailability) properties [<a href="#B35-pharmaceuticals-15-00646" rid="B35-pharmaceuticals-15-00646" class=" bibr popnode">35</a>]. </p><p>In this context, pharmacophore models can be successfully applied to filter large collections of compounds to find the so-called hits, i.e., novel molecules matching the pharmacophoric features required to be potentially active against a specific target. Since a pharmacophore does not represent exact chemical groups but chemical functionalities and their spatial relationships, the retrieved hits usually include structurally different compounds, making pharmacophores useful tools for scaffold hopping [<a href="#B7-pharmaceuticals-15-00646" rid="B7-pharmaceuticals-15-00646" class=" bibr popnode">7</a>]. </p><p>The pharmacophore-based screening can be performed directly using some of the software already mentioned for the pharmacophore generation, which allows doing this on a manually created dataset. Other useful open-access platforms for virtual screening are Pharmit (<a href="http://pharmit.csb.pitt.edu">http://pharmit.csb.pitt.edu</a>) (accessed on 20 May 2022) and ZINCPharmer (<a href="http://zincpharmer.csb.pitt.edu/">http://zincpharmer.csb.pitt.edu/</a>) (accessed on 20 May 2022). The first one allows the user to import a predefined pharmacophore query or elucidate pharmacophore and shape queries from the receptor and/or ligand structures, and to use it to screen a desired dataset among a set of provided compounds libraries or a personal one. The results are quickly computed and classified according to different criteria such as energy minimization [<a href="#B36-pharmaceuticals-15-00646" rid="B36-pharmaceuticals-15-00646" class=" bibr popnode">36</a>]. ZINCPharmer is a user-friendly online web server, which, exploiting the Pharmer research technology, allows screening of the purchasable molecules present only in the ZINC database using a pharmacophore model imported from other tools or directly derived from ligands or structures [<a href="#B37-pharmaceuticals-15-00646" rid="B37-pharmaceuticals-15-00646" class=" bibr popnode">37</a>].</p><p>In general, once the model to use has been defined, the first step in virtual screening is to consult a database that contains a large number of compounds annotated with information about the 3D structure, known target, and in some cases the purchasability. Some of the free databases include the Protein Data Bank (PDB) [<a href="#B38-pharmaceuticals-15-00646" rid="B38-pharmaceuticals-15-00646" class=" bibr popnode">38</a>], PubChem [<a href="#B39-pharmaceuticals-15-00646" rid="B39-pharmaceuticals-15-00646" class=" bibr popnode">39</a>], ChEMBL [<a href="#B21-pharmaceuticals-15-00646" rid="B21-pharmaceuticals-15-00646" class=" bibr popnode">21</a>], Zinc [<a href="#B40-pharmaceuticals-15-00646" rid="B40-pharmaceuticals-15-00646" class=" bibr popnode">40</a>], and Drugbank [<a href="#B22-pharmaceuticals-15-00646" rid="B22-pharmaceuticals-15-00646" class=" bibr popnode">22</a>]. Moreover, there are some commercially available databases such as the MDL Drug Data Report. Usually, due to the presence of a multitude of compounds in these repositories, the strategy adopted by many researchers is to filter the data by applying different parameters to reduce the computational cost of the pharmacophore searching. This may be obtained through the exclusion of compounds that are too big for the ligand pocket, the use of Lipinski&#x02019;s Rule of Five or standard metrics for lead-likeness, and removing compounds deemed to be pan-assay interference compounds (PAINS) [<a href="#B35-pharmaceuticals-15-00646" rid="B35-pharmaceuticals-15-00646" class=" bibr popnode">35</a>,<a href="#B41-pharmaceuticals-15-00646" rid="B41-pharmaceuticals-15-00646" class=" bibr popnode">41</a>]. In addition, a combination of filtering for desired pharmacological and adsorption, distribution, metabolism, excretion, and toxicological (ADMET) properties is advisable to be applied early in the virtual screening process [<a href="#B42-pharmaceuticals-15-00646" rid="B42-pharmaceuticals-15-00646" class=" bibr popnode">42</a>]. These filters should be considered suggestions, not mandatory, and they can be applied depending on the specific case. As an example, the Rule of Five is not an absolute rule, as many drugs go under the &#x0201c;beyond the rule of five&#x0201d; [<a href="#B43-pharmaceuticals-15-00646" rid="B43-pharmaceuticals-15-00646" class=" bibr popnode">43</a>]. To efficiently filter a library of compounds against such criteria, several online tools have been developed. For example, the commercial software ACD/LogD Suite (<a href="https://www.acdlabs.com/products/percepta-platform/physchem-suite/logd/">https://www.acdlabs.com/products/percepta-platform/physchem-suite/logd/</a>) (accessed on 20 May 2022) by Advanced Chemistry Development (ACD/Labs) can be used to predict ADME-related properties including hydrophobicity, lipophilicity and pKa, while Pharma Algorithms provided a suite of products via ADME Boxes (the company joined ACD/Labs and the ADME Suite is now available at <a href="https://www.acdlabs.com/products/percepta-platform/adme-suite/">https://www.acdlabs.com/products/percepta-platform/adme-suite/</a>) (accessed on 20 May 2022) that addresses issues such as solubility, oral bioavailability, absorption and distribution [<a href="#B1-pharmaceuticals-15-00646" rid="B1-pharmaceuticals-15-00646" class=" bibr popnode">1</a>]. It is necessary to underline that in some protocols reported in the literature, the evaluation of toxicity and of the ADME profile is made only after molecular docking. Therefore, it is in the choice of the researcher the use of a protocol that adds a preliminary filter, thus reducing the number of molecules on which to apply further analysis or proceed on a larger number of molecules to identify potential ligands that, if not usable directly because of their toxicity, could serve as the basis for the synthesis of new molecules. If the structure information of the target is available, the retrieved pool of compounds can be further selected by molecular docking with programs such as DOCK [<a href="#B44-pharmaceuticals-15-00646" rid="B44-pharmaceuticals-15-00646" class=" bibr popnode">44</a>] or Autodock (<a href="http://autodock.scripps.edu/">http://autodock.scripps.edu/</a>) (accessed on 20 May 2022) [<a href="#B45-pharmaceuticals-15-00646" rid="B45-pharmaceuticals-15-00646" class=" bibr popnode">45</a>], to have a preliminary evaluation of binding affinities and exclude molecules with very low values, as well as to obtain preliminary protein-ligand complexes. Molecular docking is a computational process where ligands are moved in 3D space to find a configuration of the target and ligand that maximizes the scoring function [<a href="#B1-pharmaceuticals-15-00646" rid="B1-pharmaceuticals-15-00646" class=" bibr popnode">1</a>]. With docking, therefore, ligand&#x02013;target complexes are obtained and can be evaluated more in depth, by specific methods, to verify if these compounds can be used subsequently for in vitro experimentation. One of these evaluation methods is the molecular dynamics (MD) approach. It is the pivotal theoretical approach, which can be utilized to gain molecular insight into the stability of the binding pose of the screened molecules in the active site [<a href="#B46-pharmaceuticals-15-00646" rid="B46-pharmaceuticals-15-00646" class=" bibr popnode">46</a>], giving a more accurate evaluation of their binding affinity. At the end of the virtual screening, the selected compounds matching the desired properties and giving the best results must undergo an in vitro validation process to verify if indeed the results obtained in silico are reliable. Everything said up to now represents a typical workflow followed in a virtual screening analysis and it is graphically depicted in <a href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f004/" target="figure" class="fig-table-link figpopup" rid-figpopup="pharmaceuticals-15-00646-f004" rid-ob="ob-pharmaceuticals-15-00646-f004" co-legend-rid="lgnd_pharmaceuticals-15-00646-f004"><span>Figure 4</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="pharmaceuticals-15-00646-f004" co-legend-rid="lgnd_pharmaceuticals-15-00646-f004"><a href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f004/" target="figure" rid-figpopup="pharmaceuticals-15-00646-f004" rid-ob="ob-pharmaceuticals-15-00646-f004"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140392626914240"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9145410_pharmaceuticals-15-00646-g004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pharmaceuticals-15-00646-g004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC9145410/bin/pharmaceuticals-15-00646-g004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140392626914240"><a target="object" rel="noopener" href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f004/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_pharmaceuticals-15-00646-f004"><div><a class="figpopup" href="/pmc/articles/PMC9145410/figure/pharmaceuticals-15-00646-f004/" target="figure" rid-figpopup="pharmaceuticals-15-00646-f004" rid-ob="ob-pharmaceuticals-15-00646-f004">Figure 4</a></div><!--caption a7--><div class="caption"><p>Virtual screening workflow. Basic filters and methods applicable for a virtual screening process. Scheme of the multistage vs. approach. The application of vs. follows a typical sequence of processes with a cascade of filters able to narrow down and choose a set of hits with potential biological activities.</p></div></div></div><p>Recently, MD simulations have been used to perform a thorough conformational search without any help of traditional docking procedures, with accurate all-atom force field and enhanced sampling techniques, obtaining accurate results for both binding modes and binding affinities [<a href="#B47-pharmaceuticals-15-00646" rid="B47-pharmaceuticals-15-00646" class=" bibr popnode">47</a>,<a href="#B48-pharmaceuticals-15-00646" rid="B48-pharmaceuticals-15-00646" class=" bibr popnode">48</a>].</p><p class="p p-last">Finally, it is necessary to also mention the free energy calculations used to obtain protein&#x02013;ligand binding affinity by MD simulations in conjunction with molecular docking [<a href="#B49-pharmaceuticals-15-00646" rid="B49-pharmaceuticals-15-00646" class=" bibr popnode">49</a>,<a href="#B50-pharmaceuticals-15-00646" rid="B50-pharmaceuticals-15-00646" class=" bibr popnode">50</a>,<a href="#B51-pharmaceuticals-15-00646" rid="B51-pharmaceuticals-15-00646" class=" bibr popnode">51</a>,<a href="#B52-pharmaceuticals-15-00646" rid="B52-pharmaceuticals-15-00646" class=" bibr popnode">52</a>].</p></div><div id="sec5-pharmaceuticals-15-00646" class="tsec sec"><h2 class="head no_bottom_margin" id="sec5-pharmaceuticals-15-00646title">5. Limitations and Possible Solutions</h2><p class="p p-first">As shown, pharmacophore modelling is a very useful tool for many applications in drug discovery or drug design. However, despite its strength, this method is not free of limitations that have implications in virtual screening increasing the rate of false positives and/or false negatives, and that should be in mind to find proper solutions, when possible, and to have a critical view of the results.</p><p>As explained in the previous paragraphs the quality of a pharmacophore model is strictly related to that of the input dataset and the algorithm used for the modelling. Indeed, the quality of the structures used for pharmacophore building, refinement, and validation greatly affect its reliability, and to overcome this limit many authors suggest a manual inspection of the structures used and of the annotations regarding the biological data associated with them. For what concerns the choice of the algorithm, it is important to underline that different algorithms can give different results with no overlap starting from the same macromolecular target. This could be attributed to the different screening methodology of the algorithm, but also to the specific screening dataset used among the several publicly available screening platforms [<a href="#B7-pharmaceuticals-15-00646" rid="B7-pharmaceuticals-15-00646" class=" bibr popnode">7</a>]. Therefore, an integration of different tools is recommended to cover a larger chemical space and obtain different chemotypes.</p><p>Moreover, whether the structure-based or the ligand-based approach is used, both present some intrinsic limitations. In particular, flexibility is the main drawback of the structure-based strategy. As already explained, a structure-based pharmacophore derives from the 3D structure of a macromolecule target, ideally complexed with an active compound. A single structure represents a static image reproducing a single binding mode of a ligand towards its target without taking into account the dynamic nature of the system due to receptor and ligand conformational flexibility [<a href="#B53-pharmaceuticals-15-00646" rid="B53-pharmaceuticals-15-00646" class=" bibr popnode">53</a>]. This lacking information results in pharmacophore models that may be defective in some features that could be relevant for the binding of different ligands and, therefore, in retrieving new potential hits. If experimental structures of the same protein target complexed with different ligands exist, it is possible to reduce this limit by using them to extract and merge features responsible for ligands interactions into a more refined model. However, this does not often happen and an alternative way to handle the flexibility consists in employing computational techniques such as flexible docking and/or MD simulations of the investigated system. Dynamic pharmacophores can be computed from protein-ligand MD trajectories and have been successfully applied in recent studies showing a better performance in the virtual screening of bioactive compounds compared to the classic rigid approach [<a href="#B54-pharmaceuticals-15-00646" rid="B54-pharmaceuticals-15-00646" class=" bibr popnode">54</a>,<a href="#B55-pharmaceuticals-15-00646" rid="B55-pharmaceuticals-15-00646" class=" bibr popnode">55</a>,<a href="#B56-pharmaceuticals-15-00646" rid="B56-pharmaceuticals-15-00646" class=" bibr popnode">56</a>]. Nevertheless, this methodology needs higher computational resources (a problem that is less relevant due to the increasing availability of computing resources, and in case of application to a small set of selected ligands) and expert knowledge since it produces many models that should be averaged or clustered to derive more representative pharmacophores [<a href="#B57-pharmaceuticals-15-00646" rid="B57-pharmaceuticals-15-00646" class=" bibr popnode">57</a>].</p><p>Another common problem affecting the structure-based approach is the detection of a high number of unprioritized pharmacophoric features, especially when apo-structures are the only available starting point. In this case, the derived pharmacophore model is too restrictive for virtual screening making the identification of structurally different compounds with similar characteristics difficult [<a href="#B17-pharmaceuticals-15-00646" rid="B17-pharmaceuticals-15-00646" class=" bibr popnode">17</a>]. Given this, the reduction of chemical features is crucial to obtaining a reasonable and usable model. Some strategies to apply for this purpose have already been discussed, but they depend on available data and an expert manual intervention is often required.</p><p>Regarding the ligand-based method, the major limitation is the absence of receptor complementary information related to the interaction pattern with the binding pocket or even just about its shape. By its nature, the ligand-based approach is founded on all possible chemical features and geometric information inferable from the input compounds and their flexible alignment. However, more are the chemical features to be taken into account by the algorithm, more is the time cost of it. Therefore, this approach is limited to small compounds and simple chemical features by computational/time costs, generating a less tailored pharmacophore model [<a href="#B27-pharmaceuticals-15-00646" rid="B27-pharmaceuticals-15-00646" class=" bibr popnode">27</a>].</p><p>A further complication for the ligand-based modelling is the low availability of inactive compounds, due to the lack in the literature of negative results. The use of a not enough number of molecules known as inactive for a specific protein target has an impact on the selectivity and sensitivity of the pharmacophore model obtained [<a href="#B6-pharmaceuticals-15-00646" rid="B6-pharmaceuticals-15-00646" class=" bibr popnode">6</a>].</p><p>Above the individual deficiencies, another key point to place is that the pharmacophore represents a model of the chemical features but other variables that can influence the binding of active compounds to a specific target, such as chemical solubility, cell metabolism, membrane permeability, the particular toxicologic effect could not be considered or well-integrated on it; this could have a strong impact on the real capacity of the selected compounds to have a suitable biological effect on the target [<a href="#B7-pharmaceuticals-15-00646" rid="B7-pharmaceuticals-15-00646" class=" bibr popnode">7</a>].</p><p>In general, even if some of these issues could be addressed by the use of accurate structures, training and test sets, the validity of the model obtained should be always compared to the statistical robustness of the input dataset, because no matter how complete the dataset employed could be, no model can be judged as universal [<a href="#B58-pharmaceuticals-15-00646" rid="B58-pharmaceuticals-15-00646" class=" bibr popnode">58</a>].</p><p class="p p-last">Because of the things reviewed above, the simultaneous use of both strategies, when it is allowed, is advantageous since the weaknesses of one can be balanced by the strengths of the other. Moreover, their combination with multiple approaches at different stages, such as MD simulation, docking studies and machine learning approaches, could be a valuable resource to overcome the inherent limitations of the pharmacophore modelling, improving the chance to have more complete and reliable results.</p></div><div id="sec6-pharmaceuticals-15-00646" class="tsec sec"><h2 class="head no_bottom_margin" id="sec6-pharmaceuticals-15-00646title">6. Software for Pharmacophore Modelling</h2><p class="p p-first">Several articles reported lists of software, databases and online tools for CADD applications [<a href="#B1-pharmaceuticals-15-00646" rid="B1-pharmaceuticals-15-00646" class=" bibr popnode">1</a>,<a href="#B25-pharmaceuticals-15-00646" rid="B25-pharmaceuticals-15-00646" class=" bibr popnode">25</a>,<a href="#B59-pharmaceuticals-15-00646" rid="B59-pharmaceuticals-15-00646" class=" bibr popnode">59</a>]. Moreover, online services and catalogues provide links to many resources (see <a href="https://www.click2drug.org/">https://www.click2drug.org/</a> or <a href="https://bio.tools/">https://bio.tools/</a>) (accessed on 20 May 2022). In this paragraph, we report the most used and reliable, in our opinion, and findable at the time of the writing of this review. Several programs have been developed to perform the pharmacophore modeling in an automated way using different algorithms, such as LigandScout, Schrodinger-Maestro, MOE, and Discovery Studio. Among them, LigandScout [<a href="#B60-pharmaceuticals-15-00646" rid="B60-pharmaceuticals-15-00646" class=" bibr popnode">60</a>] is widely used for pharmacophores generation from a protein&#x02013;ligand complex and performs a step-by-step ligand topology interpretation including aromatic ring detection, the assignment of functional group patterns, the determination of hybridization state and bond types, followed by ligand and binding site analysis to automatically detect and classify protein&#x02013;ligand interactions into hydrogen bonding, ionic, aromatic and lipophilic contacts according to geometric constraints. The ligand features composing this set of paired interactions define a pharmacophore model which can be also modified by removing specific features [<a href="#B60-pharmaceuticals-15-00646" rid="B60-pharmaceuticals-15-00646" class=" bibr popnode">60</a>]. The Schrodinger&#x02013;Maestro suite allows the generation of e-pharmacophores, energetically-optimized pharmacophores using the Glide XP scoring function to prioritize protein-ligands interaction sites and shows good speed and performance in virtual screening [<a href="#B61-pharmaceuticals-15-00646" rid="B61-pharmaceuticals-15-00646" class=" bibr popnode">61</a>]. MOE is an integrated computer-aided molecular design platform that offers many different services, from drug design to virtual screening and molecular simulations. Drug design by MOE allows exploiting different methods such as the structure-based design, the ligand-based design and the fragment-based discovery (<a href="http://www.chemcomp.com/MOE-Molecular_Operating_Environment.htm/">www.chemcomp.com/MOE-Molecular_Operating_Environment.htm/</a>) (accessed on 20 May 2022). Another commonly used software is Discovery Studio (BIOVIA, Dassault Syst&#x000e8;mes), which allows deriving structure-based and/or ligand-based pharmacophores. In the first case, it offers three methods to build pharmacophores: structure-based, fragment-based or receptor-ligand pharmacophores according to the available structural data. In absence of a bound ligand, this program implements the Interaction Generation protocol to create LUDI interaction maps for hydrogen and hydrophobic contacts with the binding site that can be manually selected or computationally computed. The interaction map is then converted into pharmacophore models consisting of different combinations of at least three features complementary to those of residue in the binding pocket in terms of chemical nature and location, and the resultant models are scored and ranked according to their target selectivity [<a href="#B62-pharmaceuticals-15-00646" rid="B62-pharmaceuticals-15-00646" class=" bibr popnode">62</a>]. For ligand-based pharmacophore generation, besides the use of the HipHop algorithm, DiscoveryStudio gives also the opportunity to explore the QSAR approach by the HypoGen algorithm [<a href="#B63-pharmaceuticals-15-00646" rid="B63-pharmaceuticals-15-00646" class=" bibr popnode">63</a>]. The purpose of this algorithm is to find a spatial 3D arrangement of the features shared by the training molecules whose activity on a specific target has been measured. To do so, it needs as input at least 16 compounds covering four orders of magnitudes: H-bond donors, H-bond acceptor, aromatic ring, hydrophobicity. The algorithm works in three phases: the constructive, the subtractive and the optimization. In the first phase, all the pharmacophoric conformations, which preserve at most five default features, are collected and pharmacophores preserving features shared by all the bioactive training compounds are generated; only pharmacophores fitting other active molecules are kept. In the second phase, pharmacophores fitting inactive molecules are removed; in the last, pharmacophore collection is optimized by simulating annealing algorithm. Only the pharmacophore models showing the highest score are given as output [<a href="#B7-pharmaceuticals-15-00646" rid="B7-pharmaceuticals-15-00646" class=" bibr popnode">7</a>]. Other examples of 3D QSAR programs, which generate predictive models include PHASE [<a href="#B64-pharmaceuticals-15-00646" rid="B64-pharmaceuticals-15-00646" class=" bibr popnode">64</a>], comparative molecular field analysis (CoMFA) [<a href="#B65-pharmaceuticals-15-00646" rid="B65-pharmaceuticals-15-00646" class=" bibr popnode">65</a>] and comparative similarity indices analysis (CoMSIA) [<a href="#B66-pharmaceuticals-15-00646" rid="B66-pharmaceuticals-15-00646" class=" bibr popnode">66</a>]. For a more exhaustive list, we remand to Gurung et colleagues&#x02019; review of 2021 [<a href="#B26-pharmaceuticals-15-00646" rid="B26-pharmaceuticals-15-00646" class=" bibr popnode">26</a>]. </p><p>The software tools mentioned above are commercial but there are also academic programs freely accessible. For instance, Pocket v.2 is a structure-based pharmacophore program available either on the web (<a href="http://www.pkumdl.cn:8080/pocketv2.html">http://www.pkumdl.cn:8080/pocketv2.html</a>) (accessed on 20 May 2022) or a stand-alone version that uses the Pocket module of LigBuilder (for de novo structure-based ligand design) to analyze the binding site of the target receptor by scored grids. The basic required input is the protein 3D structure in PDB format that is employed to generate pharmacophore models from receptor structures with or without a bound compound [<a href="#B67-pharmaceuticals-15-00646" rid="B67-pharmaceuticals-15-00646" class=" bibr popnode">67</a>]. </p><p>A user-friendly and freely-available web server, which exploited the common feature approach for ligand-based modelling, is represented by PharmaGist [<a href="#B68-pharmaceuticals-15-00646" rid="B68-pharmaceuticals-15-00646" class=" bibr popnode">68</a>]. The main window of PharmaGist requires only the uploading of up to 32 input molecules in mol2 format (uploadable in single or in a unique zip file), with bonds angles and length already corrected and hydrogens atoms explicitly specified, the selection of the number of output of pharmacophores that will be showed, and an e-mail address where the link to the results, stored at least for a month, will be sent. Besides this standard research PharmaGist gives also the possibility to choose some advanced options as: i. the selection of a key molecule, by this option all the other molecules will be aligned to the compound selected, otherwise all compounds are selected as key compounds; ii. the minimum number of features in pharmacophore iii. the feature weighting options, by which it is possible to modify the weight of some features (aromatic ring, hydrogen bond donor/acceptor, anion/cation and hydrophobic properties) in the scoring function; iv. User-defined features, by which additional feature types can be defined by a supported feature file format created by the user. The output consists of several tables: the first gives a summary of the input molecules&#x02019; features and the following summarized the results starting always with the highest scoring pharmacophore.</p><p class="p p-last">For those who are researching an easy to use and free available tool which exploits the QSAR method without requiring particular informatics skills, or who simply want to start learning about pharmacophore construction, the web portal 3d-qsar.com (accessed on 20 May 2022) represents a valid opportunity, being a platform hosting several applications which guide the user along the correct workflow necessary to build a pharmacophore model. The first is represented by Py-MolEdit which allows the correct compilation and uploading of the training and the test dataset in any format, giving also the possibility to draw your molecules. After using that tool, the platform gives the discretionary option to align your dataset by exploiting the Py-Align tool. At this point is possible to perform the building of a pharmacophore model with Py-CoMFA, obtained which, can be easily analyzed by Py-ConfSearch tool that performs a conformational analysis. Py-CoMFA allows the building and validation of the 3-D QSAR model with the same style as the original CoMFA software, which seems to reproduce published results with a very good agreement. The platform also provides the opportunity to make docking simulations (by Py-Docking), or to use a different QSAR algorithm (Py-ComBinE). An interface to the Protein Data Bank is also provided (by Py-PDB) [<a href="#B69-pharmaceuticals-15-00646" rid="B69-pharmaceuticals-15-00646" class=" bibr popnode">69</a>].</p></div><div id="sec7-pharmaceuticals-15-00646" class="tsec sec"><h2 class="head no_bottom_margin" id="sec7-pharmaceuticals-15-00646title">7. Examples and Case Studies</h2><p class="p p-first">Many examples in the literature present pharmacophore models obtained by QSAR and common feature pharmacophore approaches. In addition to articles and reviews published in the past that report on successful applications in pharmacophore modeling and virtual screening [<a href="#B70-pharmaceuticals-15-00646" rid="B70-pharmaceuticals-15-00646" class=" bibr popnode">70</a>,<a href="#B71-pharmaceuticals-15-00646" rid="B71-pharmaceuticals-15-00646" class=" bibr popnode">71</a>,<a href="#B72-pharmaceuticals-15-00646" rid="B72-pharmaceuticals-15-00646" class=" bibr popnode">72</a>,<a href="#B73-pharmaceuticals-15-00646" rid="B73-pharmaceuticals-15-00646" class=" bibr popnode">73</a>,<a href="#B74-pharmaceuticals-15-00646" rid="B74-pharmaceuticals-15-00646" class=" bibr popnode">74</a>], we highlight some specific example of studies.</p><p>Schuster and colleagues performed the ligand-based pharmacophore modelling for selective and nonselective 11&#x003b2;-HSD1 inhibitors. Their pharmacophore models, after a virtual screening procedure, led to the in vitro validation of 30 compounds among which seven were found to inhibit more than 70% of the activity of 11&#x003b2; -HSD1 when measured in cell lysates with IC50 values below 10 &#x000b5;M and without cytotoxicity at concentrations up to 40 &#x000b5;M [<a href="#B75-pharmaceuticals-15-00646" rid="B75-pharmaceuticals-15-00646" class=" bibr popnode">75</a>]. Pal and colleagues exploited the 3D-QSAR pharmacophore hypothesis and found relevant binding features of novel topoisomerase inhibitors [<a href="#B46-pharmaceuticals-15-00646" rid="B46-pharmaceuticals-15-00646" class=" bibr popnode">46</a>].</p><p>In a recent study, a pharmacophore model based on the experimental structure of AKT1 protein complexed with an inhibitor has been used for virtual screening on a database of natural compounds. The most promising compounds obtained by the screening have been further investigated by toxicity profile and ADMET analyses, together with molecular docking for better prediction of the potential binding [<a href="#B76-pharmaceuticals-15-00646" rid="B76-pharmaceuticals-15-00646" class=" bibr popnode">76</a>]. Experimental studies confirmed the potential inhibitory role of at least one of the selected compounds [<a href="#B76-pharmaceuticals-15-00646" rid="B76-pharmaceuticals-15-00646" class=" bibr popnode">76</a>] and, in addition, its synergic effect with quercetin to inhibition of cell growth and induce apoptosis [<a href="#B77-pharmaceuticals-15-00646" rid="B77-pharmaceuticals-15-00646" class=" bibr popnode">77</a>].</p><p class="p p-last">Due to the pandemic, many research teams explored the opportunity to find novel drugs through time-saving CADD approaches. This is the case of a study oriented to finding novel inhibitors of TMPRSS2, or type II transmembrane serine protease, a human protein involved in the activating processes of SARS-CoV-2 [<a href="#B78-pharmaceuticals-15-00646" rid="B78-pharmaceuticals-15-00646" class=" bibr popnode">78</a>]. TMPRSS2 is of particular interest for its involvement also in cancer pathologies. The in silico screening has been performed by a pharmacophore-based approach, starting from camostat mesylate, a known inhibitor of serine protease 2, also approved for drug. The authors selected 10 pharmacophoric features of camostat mesylate and selected 2140 compounds from a public database containing more than 30,000 natural compounds. After a molecular docking analysis, the 2140 compounds were filtered to 85 candidates with docking scores similar to or better than the known inhibitor. The computational study has been further extended to analyze the list of filtered compounds for matching with Lipinski&#x02019;s rule of five and for ADMET properties. The study has been cited many times in literature, due to the interest in the results and the need to verify by experimental approaches the real potential inhibitory activity of filtered compounds.</p></div><div id="sec8-pharmaceuticals-15-00646" class="tsec sec"><h2 class="head no_bottom_margin" id="sec8-pharmaceuticals-15-00646title">8. Future Perspectives</h2><p class="p p-first-last">CADD approaches have been successfully developed to improve the ability to discover new drugs. The increasing computational power of hardware opens the perspective of many more applications, also by increasing the availability of protein target structures through deep learning and AI-based prediction tools [<a href="#B79-pharmaceuticals-15-00646" rid="B79-pharmaceuticals-15-00646" class=" bibr popnode">79</a>]. The deep learning approaches have been also recently applied to drug-target interaction [<a href="#B80-pharmaceuticals-15-00646" rid="B80-pharmaceuticals-15-00646" class=" bibr popnode">80</a>]. However, the experimental steps needed to validate the compound activity and safety remain time-consuming and will always slow down the whole drug discovery process. From this point of view, the pharmacophore modelling followed by the virtual screening of databases of approved drugs, in the drug repurposing perspective, represents the most advanced rational approach, and the most promising way to find novel therapies.</p></div><div id="funding-group-a.g.b.s" class="tsec sec"><h2 class="head no_bottom_margin" id="funding-group-a.g.b.stitle">Funding Statement</h2><p>This research received no external funding. DG is supported by a post-doc fellowship under the project framework &#x0201c;CIR01_00017-&#x0201c;CNRBiOmics&#x02013;Centro Nazionale di Ricerca in Bioinformatica per le Scienze &#x0201c;Omiche&#x0201d;&#x02013;Rafforzamento del capitale umano&#x0201d; funded by MUR, CUP B56J20000960001.</p></div><div id="notes-a.i.b" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.i.btitle">Author Contributions</h2><p>All authors contributed to conceptualization, writing&#x02014;review and editing. All authors have read and agreed to the published version of the manuscript.</p></div><div id="notes-a.i.c" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.i.ctitle">Institutional Review Board Statement</h2><p>Not applicable.</p></div><div id="notes-a.i.d" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.i.dtitle">Informed Consent Statement</h2><p>Not applicable.</p></div><div id="notes-a.i.e" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.i.etitle">Data Availability Statement</h2><p>No new data were created or analyzed in this study. Data sharing is not applicable to this article.</p></div><div id="notes-a.i.f" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.i.ftitle">Conflicts of Interest</h2><p>The authors declare no conflict of interest.</p></div><div id="fn-group-a.i.a" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.i.atitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn-a.i.a.a"><p class="p p-first-last"><strong>Publisher&#x02019;s Note:</strong> MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></p></div></div><div id="ref-list-a.i.g" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.i.gtitle">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1-pharmaceuticals-15-00646">1. <span class="element-citation">Song C.M., Lim S.J., Tong J.C. Recent Advances in Computer-Aided Drug Design. <span><span class="ref-journal">Brief. Bioinform. </span>2009;<span class="ref-vol">10</span>:579–591. doi: 10.1093/bib/bbp023.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19433475" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbib%2Fbbp023" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brief.+Bioinform.&amp;title=Recent+Advances+in+Computer-Aided+Drug+Design&amp;author=C.M.+Song&amp;author=S.J.+Lim&amp;author=J.C.+Tong&amp;volume=10&amp;publication_year=2009&amp;pages=579-591&amp;pmid=19433475&amp;doi=10.1093/bib/bbp023&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2-pharmaceuticals-15-00646">2. <span class="element-citation">Voet A., Banwell E.F., Sahu K.K., Heddle J.G., Zhang K.Y.J. Protein Interface Pharmacophore Mapping Tools for Small Molecule Protein: Protein Interaction Inhibitor Discovery. <span><span class="ref-journal">Curr. Top. Med. Chem. </span>2013;<span class="ref-vol">3</span>:989–1001. doi: 10.2174/1568026611313090003.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23651479" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2174%2F1568026611313090003" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Top.+Med.+Chem.&amp;title=Protein+Interface+Pharmacophore+Mapping+Tools+for+Small+Molecule+Protein:+Protein+Interaction+Inhibitor+Discovery&amp;author=A.+Voet&amp;author=E.F.+Banwell&amp;author=K.K.+Sahu&amp;author=J.G.+Heddle&amp;author=K.Y.J.+Zhang&amp;volume=3&amp;publication_year=2013&amp;pages=989-1001&amp;pmid=23651479&amp;doi=10.2174/1568026611313090003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3-pharmaceuticals-15-00646">3. <span class="element-citation">Seidel T., Schuetz D.A., Garon A., Langer T. The Pharmacophore Concept and Its Applications in Computer-Aided Drug Design. <span><span class="ref-journal">Prog. Chem. Org. Nat. Prod. </span>2019;<span class="ref-vol">110</span>:99–141.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31621012" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prog.+Chem.+Org.+Nat.+Prod.&amp;title=The+Pharmacophore+Concept+and+Its+Applications+in+Computer-Aided+Drug+Design&amp;author=T.+Seidel&amp;author=D.A.+Schuetz&amp;author=A.+Garon&amp;author=T.+Langer&amp;volume=110&amp;publication_year=2019&amp;pages=99-141&amp;pmid=31621012&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4-pharmaceuticals-15-00646">4. <span class="element-citation">Seidel T., Wieder O., Garon A., Langer T. Applications of the Pharmacophore Concept in Natural Product Inspired Drug Design. <span><span class="ref-journal">Mol. Inform. </span>2020;<span class="ref-vol">39</span>:1–11. doi: 10.1002/minf.202000059.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7685156/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32578959" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fminf.202000059" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Inform.&amp;title=Applications+of+the+Pharmacophore+Concept+in+Natural+Product+Inspired+Drug+Design&amp;author=T.+Seidel&amp;author=O.+Wieder&amp;author=A.+Garon&amp;author=T.+Langer&amp;volume=39&amp;publication_year=2020&amp;pages=1-11&amp;doi=10.1002/minf.202000059&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5-pharmaceuticals-15-00646">5. <span class="element-citation">Wermuth C.G., Ganellin C.R., Lindberg P., Mitscher L.A. Glossary of terms used in medicinal chemistry. <span><span class="ref-journal">Pure Appl. Chem. </span>1998;<span class="ref-vol">70</span>:1129–1142. doi: 10.1351/pac199870051129.</span> [<a href="//doi.org/10.1351%2Fpac199870051129" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pure+Appl.+Chem.&amp;title=Glossary+of+terms+used+in+medicinal+chemistry&amp;author=C.G.+Wermuth&amp;author=C.R.+Ganellin&amp;author=P.+Lindberg&amp;author=L.A.+Mitscher&amp;volume=70&amp;publication_year=1998&amp;pages=1129-1142&amp;doi=10.1351/pac199870051129&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6-pharmaceuticals-15-00646">6. <span class="element-citation">Kaserer T., Beck K.R., Akram M., Odermatt A., Schuster D., Willett P. Pharmacophore Models and Pharmacophore-Based Virtual Screening: Concepts and Applications Exemplified on Hydroxysteroid Dehydrogenases. <span><span class="ref-journal">Molecules. </span>2015;<span class="ref-vol">20</span>:22799–22832. doi: 10.3390/molecules201219880.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6332202/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26703541" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fmolecules201219880" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Pharmacophore+Models+and+Pharmacophore-Based+Virtual+Screening:+Concepts+and+Applications+Exemplified+on+Hydroxysteroid+Dehydrogenases&amp;author=T.+Kaserer&amp;author=K.R.+Beck&amp;author=M.+Akram&amp;author=A.+Odermatt&amp;author=D.+Schuster&amp;volume=20&amp;publication_year=2015&amp;pages=22799-22832&amp;pmid=26703541&amp;doi=10.3390/molecules201219880&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7-pharmaceuticals-15-00646">7. <span class="element-citation">Vuorinen A., Schuster D. Methods for generating and applying pharmacophore models as virtual screening filters and for bioactivity profiling. <span><span class="ref-journal">Methods. </span>2015;<span class="ref-vol">71</span>:113–134. doi: 10.1016/j.ymeth.2014.10.013.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25461773" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ymeth.2014.10.013" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Methods+for+generating+and+applying+pharmacophore+models+as+virtual+screening+filters+and+for+bioactivity+profiling&amp;author=A.+Vuorinen&amp;author=D.+Schuster&amp;volume=71&amp;publication_year=2015&amp;pages=113-134&amp;pmid=25461773&amp;doi=10.1016/j.ymeth.2014.10.013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8-pharmaceuticals-15-00646">8. <span class="element-citation">Sanders M.P.A., McGuire R., Roumen L., de Esch I.J.P., de Vlieg J., Klomp J.P.G., de Graaf C. From the Protein&#x02019;s Perspective: The Benefits and Challenges of Protein Structure-Based Pharmacophore Modeling. <span><span class="ref-journal">MedChemComm. </span>2012;<span class="ref-vol">3</span>:28–38. doi: 10.1039/C1MD00210D.</span> [<a href="//doi.org/10.1039%2FC1MD00210D" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=MedChemComm&amp;title=From+the+Protein&#x02019;s+Perspective:+The+Benefits+and+Challenges+of+Protein+Structure-Based+Pharmacophore+Modeling&amp;author=M.P.A.+Sanders&amp;author=R.+McGuire&amp;author=L.+Roumen&amp;author=I.J.P.+de+Esch&amp;author=J.+de+Vlieg&amp;volume=3&amp;publication_year=2012&amp;pages=28-38&amp;doi=10.1039/C1MD00210D&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9-pharmaceuticals-15-00646">9. <span class="element-citation">Chandrasekaran B., Agrawal N., Kaushik S.  <span class="ref-journal">Encyclopedia of Bioinformatics and Computational Biology: ABC of Bioinformatics.</span> 1&#x02013;3. Elsevier; Amsterdam, The Netherlands: 2018. Pharmacophore Development; pp. 677–687. [<a href="//doi.org/10.1016%2FB978-0-12-809633-8.20276-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Encyclopedia+of+Bioinformatics+and+Computational+Biology:+ABC+of+Bioinformatics&amp;author=B.+Chandrasekaran&amp;author=N.+Agrawal&amp;author=S.+Kaushik&amp;publication_year=2018&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10-pharmaceuticals-15-00646">10. <span class="element-citation">Muhammed M.T., Aki-Yalcin E. Homology modeling in drug discovery: Overview, current applications, and future perspectives. <span><span class="ref-journal">Chem. Biol. Drug Des. </span>2019;<span class="ref-vol">93</span>:12–20. doi: 10.1111/cbdd.13388.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30187647" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fcbdd.13388" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem.+Biol.+Drug+Des.&amp;title=Homology+modeling+in+drug+discovery:+Overview,+current+applications,+and+future+perspectives&amp;author=M.T.+Muhammed&amp;author=E.+Aki-Yalcin&amp;volume=93&amp;publication_year=2019&amp;pages=12-20&amp;pmid=30187647&amp;doi=10.1111/cbdd.13388&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11-pharmaceuticals-15-00646">11. <span class="element-citation">Jumper J., Evans R., Pritzel A., Green T., Figurnov M., Ronneberger O., Tunyasuvunakool K., Bates R., &#x0017d;&#x000ed;dek A., Potapenko A., et al. Highly accurate protein structure prediction with AlphaFold. <span><span class="ref-journal">Nature. </span>2021;<span class="ref-vol">596</span>:583–589. doi: 10.1038/s41586-021-03819-2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8371605/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-021-03819-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly+accurate+protein+structure+prediction+with+AlphaFold&amp;author=J.+Jumper&amp;author=R.+Evans&amp;author=A.+Pritzel&amp;author=T.+Green&amp;author=M.+Figurnov&amp;volume=596&amp;publication_year=2021&amp;pages=583-589&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12-pharmaceuticals-15-00646">12. <span class="element-citation">Sousa S.F., Fernandes P.A., Ramos M.J. Protein-ligand docking: Current status and future challenges. <span><span class="ref-journal">Proteins. </span>2006;<span class="ref-vol">65</span>:15–26. doi: 10.1002/prot.21082.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16862531" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fprot.21082" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Protein-ligand+docking:+Current+status+and+future+challenges&amp;author=S.F.+Sousa&amp;author=P.A.+Fernandes&amp;author=M.J.+Ramos&amp;volume=65&amp;publication_year=2006&amp;pages=15-26&amp;pmid=16862531&amp;doi=10.1002/prot.21082&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13-pharmaceuticals-15-00646">13. <span class="element-citation">Xie Z.R., Hwang M.J. Methods for predicting protein-ligand binding sites. <span><span class="ref-journal">Methods Mol. Biol. </span>2015;<span class="ref-vol">1215</span>:383–398.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25330972" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol.+Biol.&amp;title=Methods+for+predicting+protein-ligand+binding+sites&amp;author=Z.R.+Xie&amp;author=M.J.+Hwang&amp;volume=1215&amp;publication_year=2015&amp;pages=383-398&amp;pmid=25330972&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14-pharmaceuticals-15-00646">14. <span class="element-citation">Goodford P.J. A computational procedure for determining energetically favorable binding sites on biologically important macromolecules. <span><span class="ref-journal">J. Med. Chem. </span>1985;<span class="ref-vol">28</span>:849–857. doi: 10.1021/jm00145a002.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3892003" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fjm00145a002" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=A+computational+procedure+for+determining+energetically+favorable+binding+sites+on+biologically+important+macromolecules&amp;author=P.J.+Goodford&amp;volume=28&amp;publication_year=1985&amp;pages=849-857&amp;pmid=3892003&amp;doi=10.1021/jm00145a002&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15-pharmaceuticals-15-00646">15. <span class="element-citation">B&#x000f6;hm H.J. The computer program LUDI: A new method for the de novo design of enzyme inhibitors. <span><span class="ref-journal">J. Comput. Aided Mol. Des. </span>1992;<span class="ref-vol">6</span>:61–78. doi: 10.1007/BF00124387.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1583540" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2FBF00124387" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Aided+Mol.+Des.&amp;title=The+computer+program+LUDI:+A+new+method+for+the+de+novo+design+of+enzyme+inhibitors&amp;author=H.J.+B&#x000f6;hm&amp;volume=6&amp;publication_year=1992&amp;pages=61-78&amp;pmid=1583540&amp;doi=10.1007/BF00124387&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16-pharmaceuticals-15-00646">16. <span class="element-citation">Choudhury C., Narahari Sastry G. Pharmacophore Modelling and Screening: Concepts, Recent Developments and Applications in Rational Drug Design. <span><span class="ref-journal">Chall. Adv. Comput. Chem. Phys. </span>2019;<span class="ref-vol">27</span>:25–53.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chall.+Adv.+Comput.+Chem.+Phys.&amp;title=Pharmacophore+Modelling+and+Screening:+Concepts,+Recent+Developments+and+Applications+in+Rational+Drug+Design&amp;author=C.+Choudhury&amp;author=G.+Narahari+Sastry&amp;volume=27&amp;publication_year=2019&amp;pages=25-53&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17-pharmaceuticals-15-00646">17. <span class="element-citation">Yang S.Y. Pharmacophore modeling and applications in drug discovery: Challenges and recent advances. <span><span class="ref-journal">Drug Discov. Today. </span>2010;<span class="ref-vol">15</span>:444–450. doi: 10.1016/j.drudis.2010.03.013.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20362693" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.drudis.2010.03.013" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Discov.+Today&amp;title=Pharmacophore+modeling+and+applications+in+drug+discovery:+Challenges+and+recent+advances&amp;author=S.Y.+Yang&amp;volume=15&amp;publication_year=2010&amp;pages=444-450&amp;pmid=20362693&amp;doi=10.1016/j.drudis.2010.03.013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18-pharmaceuticals-15-00646">18. <span class="element-citation">Huang N., Shoichet B.K., Irwin J.J. Benchmarking Sets for Molecular Docking. <span><span class="ref-journal">J. Med. Chem. </span>2006;<span class="ref-vol">49</span>:6789–6801. doi: 10.1021/jm0608356.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3383317/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17154509" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fjm0608356" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=Benchmarking+Sets+for+Molecular+Docking&amp;author=N.+Huang&amp;author=B.K.+Shoichet&amp;author=J.J.+Irwin&amp;volume=49&amp;publication_year=2006&amp;pages=6789-6801&amp;pmid=17154509&amp;doi=10.1021/jm0608356&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19-pharmaceuticals-15-00646">19. <span class="element-citation">Mysinger M.M., Carchia M., Irwin J.J., Shoichet B.K. Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking. <span><span class="ref-journal">J. Med. Chem. </span>2012;<span class="ref-vol">55</span>:6582–6594. doi: 10.1021/jm300687e.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3405771/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22716043" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fjm300687e" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=Directory+of+Useful+Decoys,+Enhanced+(DUD-E):+Better+Ligands+and+Decoys+for+Better+Benchmarking&amp;author=M.M.+Mysinger&amp;author=M.+Carchia&amp;author=J.J.+Irwin&amp;author=B.K.+Shoichet&amp;volume=55&amp;publication_year=2012&amp;pages=6582-6594&amp;pmid=22716043&amp;doi=10.1021/jm300687e&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20-pharmaceuticals-15-00646">20. <span class="element-citation">Chichester C., Digles D., Siebes R., Loizou A., Groth P., Harland L. Drug discovery FAQs: Workflows for answering multidomain drug discovery questions. <span><span class="ref-journal">Drug Discov. Today. </span>2015;<span class="ref-vol">20</span>:399–405. doi: 10.1016/j.drudis.2014.11.006.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25463038" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.drudis.2014.11.006" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Discov.+Today&amp;title=Drug+discovery+FAQs:+Workflows+for+answering+multidomain+drug+discovery+questions&amp;author=C.+Chichester&amp;author=D.+Digles&amp;author=R.+Siebes&amp;author=A.+Loizou&amp;author=P.+Groth&amp;volume=20&amp;publication_year=2015&amp;pages=399-405&amp;pmid=25463038&amp;doi=10.1016/j.drudis.2014.11.006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21-pharmaceuticals-15-00646">21. <span class="element-citation">Gaulton A., Hersey A., Nowotka M., Bento A.P., Chambers J., Mendez D., Mutowo P., Atkinson F., Bellis L.J., Cibri&#x000e1;n-Uhalte E., et al. The ChEMBL Database in 2017. <span><span class="ref-journal">Nucleic Acids Res. </span>2017;<span class="ref-vol">45</span>:D945–D954. doi: 10.1093/nar/gkw1074.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5210557/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27899562" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkw1074" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+ChEMBL+Database+in+2017&amp;author=A.+Gaulton&amp;author=A.+Hersey&amp;author=M.+Nowotka&amp;author=A.P.+Bento&amp;author=J.+Chambers&amp;volume=45&amp;publication_year=2017&amp;pages=D945-D954&amp;pmid=27899562&amp;doi=10.1093/nar/gkw1074&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22-pharmaceuticals-15-00646">22. <span class="element-citation">Wishart D.S., Knox C., Guo A.C., Cheng D., Shrivastava S., Tzur D., Gautam B., Hassanali M. DrugBank: A knowledgebase for drugs, drug actions and drug targets. <span><span class="ref-journal">Nucleic Acids Res. </span>2008;<span class="ref-vol">36</span>:D901–D906. doi: 10.1093/nar/gkm958.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2238889/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18048412" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkm958" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=DrugBank:+A+knowledgebase+for+drugs,+drug+actions+and+drug+targets&amp;author=D.S.+Wishart&amp;author=C.+Knox&amp;author=A.C.+Guo&amp;author=D.+Cheng&amp;author=S.+Shrivastava&amp;volume=36&amp;publication_year=2008&amp;pages=D901-D906&amp;pmid=18048412&amp;doi=10.1093/nar/gkm958&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23-pharmaceuticals-15-00646">23. <span class="element-citation">Wang Y., Bryant S.H., Cheng T., Wang J., Gindulyte A., Shoemaker B.A., Thiessen P.A., He S., Zhang J. PubChem BioAssay: 2017 update. <span><span class="ref-journal">Nucleic Acids Res. </span>2017;<span class="ref-vol">45</span>:D955–D963. doi: 10.1093/nar/gkw1118.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5210581/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27899599" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkw1118" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=PubChem+BioAssay:+2017+update&amp;author=Y.+Wang&amp;author=S.H.+Bryant&amp;author=T.+Cheng&amp;author=J.+Wang&amp;author=A.+Gindulyte&amp;volume=45&amp;publication_year=2017&amp;pages=D955-D963&amp;pmid=27899599&amp;doi=10.1093/nar/gkw1118&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24-pharmaceuticals-15-00646">24. <span class="element-citation">Thomas R.S., Paules R.S., Simeonov A., Fitzpatrick S.C., Crofton K.M., Casey W.M., Mendrick D.L. The US Federal Tox21 Program: A strategic and operational plan for continued leadership. <span><span class="ref-journal">ALTEX. </span>2018;<span class="ref-vol">35</span>:163–168. doi: 10.14573/altex.1803011.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6664816/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29529324" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.14573%2Faltex.1803011" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ALTEX&amp;title=The+US+Federal+Tox21+Program:+A+strategic+and+operational+plan+for+continued+leadership&amp;author=R.S.+Thomas&amp;author=R.S.+Paules&amp;author=A.+Simeonov&amp;author=S.C.+Fitzpatrick&amp;author=K.M.+Crofton&amp;volume=35&amp;publication_year=2018&amp;pages=163-168&amp;pmid=29529324&amp;doi=10.14573/altex.1803011&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25-pharmaceuticals-15-00646">25. <span class="element-citation">Gurung A.B., Ali M.A., Lee J., Farah M.A., Al-Anazi K.M. An Updated Review of Computer-Aided Drug Design and Its Application to COVID-19. <span><span class="ref-journal">Biomed. Res. Int. </span>2021;<span class="ref-vol">2021</span>:8853056. doi: 10.1155/2021/8853056.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8241505/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34258282" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1155%2F2021%2F8853056" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biomed.+Res.+Int.&amp;title=An+Updated+Review+of+Computer-Aided+Drug+Design+and+Its+Application+to+COVID-19&amp;author=A.B.+Gurung&amp;author=M.A.+Ali&amp;author=J.+Lee&amp;author=M.A.+Farah&amp;author=K.M.+Al-Anazi&amp;volume=2021&amp;publication_year=2021&amp;pages=8853056&amp;pmid=34258282&amp;doi=10.1155/2021/8853056&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26-pharmaceuticals-15-00646">26. <span class="element-citation">Seidel T., Ibis G., Bendix F., Wolber G. Strategies for 3D pharmacophore-based virtual screening. <span><span class="ref-journal">Drug Discov. Today Technol. </span>2010;<span class="ref-vol">7</span>:e221–e228. doi: 10.1016/j.ddtec.2010.11.004.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24103798" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ddtec.2010.11.004" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Discov.+Today+Technol.&amp;title=Strategies+for+3D+pharmacophore-based+virtual+screening&amp;author=T.+Seidel&amp;author=G.+Ibis&amp;author=F.+Bendix&amp;author=G.+Wolber&amp;volume=7&amp;publication_year=2010&amp;pages=e221-e228&amp;doi=10.1016/j.ddtec.2010.11.004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27-pharmaceuticals-15-00646">27. <span class="element-citation">Wolber G., Dornhofer A.A., Langer T. Efficient overlay of small organic molecules using 3D pharmacophores. <span><span class="ref-journal">J. Comput. Aided Mol. Des. </span>2006;<span class="ref-vol">20</span>:773–788. doi: 10.1007/s10822-006-9078-7.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17051340" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs10822-006-9078-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Aided+Mol.+Des.&amp;title=Efficient+overlay+of+small+organic+molecules+using+3D+pharmacophores&amp;author=G.+Wolber&amp;author=A.A.+Dornhofer&amp;author=T.+Langer&amp;volume=20&amp;publication_year=2006&amp;pages=773-788&amp;pmid=17051340&amp;doi=10.1007/s10822-006-9078-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28-pharmaceuticals-15-00646">28. <span class="element-citation">Tyagi R., Singh A., Chaudhary K.K.  Pharmacophore modeling and its application. In: Singh D.B., Pathak R.K., editors. <span class="ref-journal">Bioinformatics: Methods and Applications.</span> Academic Press; Cambridge, MA, USA: 2021.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Bioinformatics:+Methods+and+Applications&amp;author=R.+Tyagi&amp;author=A.+Singh&amp;author=K.K.+Chaudhary&amp;publication_year=2021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29-pharmaceuticals-15-00646">29. <span class="element-citation">Barnum D., Greene J., Smellie A., Sprague P. Identification of common functional configurations among molecules. <span><span class="ref-journal">J. Chem. Inf. Comput. Sci. </span>1996;<span class="ref-vol">36</span>:563–571. doi: 10.1021/ci950273r.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8690757" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fci950273r" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Comput.+Sci.&amp;title=Identification+of+common+functional+configurations+among+molecules&amp;author=D.+Barnum&amp;author=J.+Greene&amp;author=A.+Smellie&amp;author=P.+Sprague&amp;volume=36&amp;publication_year=1996&amp;pages=563-571&amp;pmid=8690757&amp;doi=10.1021/ci950273r&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30-pharmaceuticals-15-00646">30. <span class="element-citation">Prieto-Mart&#x000ed;nez F.D., Norinder U., Medina-Franco J.L.  Cheminformatics Explorations of Natural Products. In: Kinghorn A.D., Gibbons H.F.S., Kobayashi J., Asakawa Y., Liu J.K., editors. <span class="ref-journal">Progress in the Chemistry of Organic Natural Products.</span> Springer; Cham, Switzerland: 2019.  [<a href="https://pubmed.ncbi.nlm.nih.gov/31621009" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Progress+in+the+Chemistry+of+Organic+Natural+Products&amp;author=F.D.+Prieto-Mart&#x000ed;nez&amp;author=U.+Norinder&amp;author=J.L.+Medina-Franco&amp;publication_year=2019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31-pharmaceuticals-15-00646">31. <span class="element-citation">Lill M.A. Multi-dimensional QSAR in drug discovery. <span><span class="ref-journal">Drug Discov. Today. </span>2007;<span class="ref-vol">12</span>:1013–1017. doi: 10.1016/j.drudis.2007.08.004.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18061879" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.drudis.2007.08.004" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Discov.+Today&amp;title=Multi-dimensional+QSAR+in+drug+discovery&amp;author=M.A.+Lill&amp;volume=12&amp;publication_year=2007&amp;pages=1013-1017&amp;pmid=18061879&amp;doi=10.1016/j.drudis.2007.08.004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32-pharmaceuticals-15-00646">32. <span class="element-citation">Lu S.-H., Wu J.W., Liu H.-L., Zhao J.-H., Liu K.-T., Chuang C.-K., Lin H.-Y., Tsai W.-B., Ho Y. The discovery of potential acetylcholinesterase inhibitors: A combination of pharmacophore modeling, virtual screening, and molecular docking studies. <span><span class="ref-journal">J. Biomed. Sci. </span>2011;<span class="ref-vol">18</span>:8.  doi: 10.1186/1423-0127-18-8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3036604/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21251245" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1423-0127-18-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biomed.+Sci.&amp;title=The+discovery+of+potential+acetylcholinesterase+inhibitors:+A+combination+of+pharmacophore+modeling,+virtual+screening,+and+molecular+docking+studies&amp;author=S.-H.+Lu&amp;author=J.W.+Wu&amp;author=H.-L.+Liu&amp;author=J.-H.+Zhao&amp;author=K.-T.+Liu&amp;volume=18&amp;publication_year=2011&amp;pages=8&amp;pmid=21251245&amp;doi=10.1186/1423-0127-18-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33-pharmaceuticals-15-00646">33. <span class="element-citation">Triballeau N., Acher F., Brabet I. Virtual Screening Workflow Development Guided by the &#x0201c;Receiver Operating Characteristic&#x0201d; Curve Approach. Application to High-Throughput Docking on Metabotropic Glutamate Receptor Subtype 4. <span><span class="ref-journal">J. Med. Chem. </span>2005;<span class="ref-vol">48</span>:2534–2547. doi: 10.1021/jm049092j.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15801843" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fjm049092j" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=Virtual+Screening+Workflow+Development+Guided+by+the+&#x0201c;Receiver+Operating+Characteristic&#x0201d;+Curve+Approach.+Application+to+High-Throughput+Docking+on+Metabotropic+Glutamate+Receptor+Subtype+4&amp;author=N.+Triballeau&amp;author=F.+Acher&amp;author=I.+Brabet&amp;volume=48&amp;publication_year=2005&amp;pages=2534-2547&amp;pmid=15801843&amp;doi=10.1021/jm049092j&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34-pharmaceuticals-15-00646">34. <span class="element-citation">Mitra I., Saha A., Roy K. Pharmacophore mapping of arylamino-substituted benzo[b]thiophenes as free radical scavengers. <span><span class="ref-journal">J. Mol. Model. </span>2010;<span class="ref-vol">16</span>:1585–1596. doi: 10.1007/s00894-010-0661-4.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20195668" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00894-010-0661-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Model.&amp;title=Pharmacophore+mapping+of+arylamino-substituted+benzo[b]thiophenes+as+free+radical+scavengers&amp;author=I.+Mitra&amp;author=A.+Saha&amp;author=K.+Roy&amp;volume=16&amp;publication_year=2010&amp;pages=1585-1596&amp;pmid=20195668&amp;doi=10.1007/s00894-010-0661-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35-pharmaceuticals-15-00646">35. <span class="element-citation">Maia E.H.B., Assis L.C., de Oliveira T.A., da Silva A.M., Taranto A.G. Structure-Based Virtual Screening: From Classical to Artificial Intelligence. <span><span class="ref-journal">Front. Chem. </span>2020;<span class="ref-vol">8</span>:343.  doi: 10.3389/fchem.2020.00343.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7200080/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32411671" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffchem.2020.00343" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Chem.&amp;title=Structure-Based+Virtual+Screening:+From+Classical+to+Artificial+Intelligence&amp;author=E.H.B.+Maia&amp;author=L.C.+Assis&amp;author=T.A.+de+Oliveira&amp;author=A.M.+da+Silva&amp;author=A.G.+Taranto&amp;volume=8&amp;publication_year=2020&amp;pages=343&amp;pmid=32411671&amp;doi=10.3389/fchem.2020.00343&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36-pharmaceuticals-15-00646">36. <span class="element-citation">Sunseri J., Koes D.R. Pharmit: Interactive exploration of chemical space. <span><span class="ref-journal">Nucleic Acids Res. </span>2016;<span class="ref-vol">44</span>:W442–W448. doi: 10.1093/nar/gkw287.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4987880/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27095195" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkw287" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Pharmit:+Interactive+exploration+of+chemical+space&amp;author=J.+Sunseri&amp;author=D.R.+Koes&amp;volume=44&amp;publication_year=2016&amp;pages=W442-W448&amp;pmid=27095195&amp;doi=10.1093/nar/gkw287&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37-pharmaceuticals-15-00646">37. <span class="element-citation">Koes D.R., Camacho C.J. ZINCPharmer: Pharmacophore search of the ZINC database. <span><span class="ref-journal">Nucleic Acids Res. </span>2012;<span class="ref-vol">40</span>:W409–W414. doi: 10.1093/nar/gks378.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3394271/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22553363" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgks378" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=ZINCPharmer:+Pharmacophore+search+of+the+ZINC+database&amp;author=D.R.+Koes&amp;author=C.J.+Camacho&amp;volume=40&amp;publication_year=2012&amp;pages=W409-W414&amp;pmid=22553363&amp;doi=10.1093/nar/gks378&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38-pharmaceuticals-15-00646">38. <span class="element-citation">Burley S.K., Bhikadiya C., Bi C., Bittrich S., Chen L., Crichlow G.V., Christie C.H., Dalenberg K., Di Costanzo L., Duarte J.M., et al. RCSB Protein Data Bank: Powerful new tools for exploring 3D structures of biological macromolecules for basic and applied research and education in fundamental biology, biomedicine, biotechnology, bioengineering and energy sciences. <span><span class="ref-journal">Nucleic Acids Res. </span>2021;<span class="ref-vol">49</span>:D437–D451. doi: 10.1093/nar/gkaa1038.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7779003/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33211854" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkaa1038" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=RCSB+Protein+Data+Bank:+Powerful+new+tools+for+exploring+3D+structures+of+biological+macromolecules+for+basic+and+applied+research+and+education+in+fundamental+biology,+biomedicine,+biotechnology,+bioengineering+and+energy+sciences&amp;author=S.K.+Burley&amp;author=C.+Bhikadiya&amp;author=C.+Bi&amp;author=S.+Bittrich&amp;author=L.+Chen&amp;volume=49&amp;publication_year=2021&amp;pages=D437-D451&amp;pmid=33211854&amp;doi=10.1093/nar/gkaa1038&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39-pharmaceuticals-15-00646">39. <span class="element-citation">Kim S., Chen J., Cheng T., Gindulyte A., He J., He S., Li Q., Shoemaker B.A., Thiessen P.A., Yu B., et al. PubChem in 2021: New data content and improved web interfaces. <span><span class="ref-journal">Nucleic Acids Res. </span>2021;<span class="ref-vol">49</span>:D1388–D1395. doi: 10.1093/nar/gkaa971.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7778930/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33151290" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkaa971" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=PubChem+in+2021:+New+data+content+and+improved+web+interfaces&amp;author=S.+Kim&amp;author=J.+Chen&amp;author=T.+Cheng&amp;author=A.+Gindulyte&amp;author=J.+He&amp;volume=49&amp;publication_year=2021&amp;pages=D1388-D1395&amp;pmid=33151290&amp;doi=10.1093/nar/gkaa971&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40-pharmaceuticals-15-00646">40. <span class="element-citation">Irwin J.J., Shoichet B.K. ZINC&#x02014;A Free Database of Commercially Available Compounds for Virtual Screening. <span><span class="ref-journal">J. Chem. Inf. Model. </span>2005;<span class="ref-vol">45</span>:177–182. doi: 10.1021/ci049714+.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1360656/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15667143" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fci049714%2B" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Model.&amp;title=ZINC&#x02014;A+Free+Database+of+Commercially+Available+Compounds+for+Virtual+Screening&amp;author=J.J.+Irwin&amp;author=B.K.+Shoichet&amp;volume=45&amp;publication_year=2005&amp;pages=177-182&amp;pmid=15667143&amp;doi=10.1021/ci049714+&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41-pharmaceuticals-15-00646">41. <span class="element-citation">Carpenter K.A., Huang X. Machine Learning-Based Virtual Screening and Its Applications to Alzheimer&#x02019;s Drug Discovery: A Review. <span><span class="ref-journal">Curr. Pharm. Des. </span>2018;<span class="ref-vol">24</span>:3347–3358. doi: 10.2174/1381612824666180607124038.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6327115/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29879881" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2174%2F1381612824666180607124038" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Pharm.+Des.&amp;title=Machine+Learning-Based+Virtual+Screening+and+Its+Applications+to+Alzheimer&#x02019;s+Drug+Discovery:+A+Review&amp;author=K.A.+Carpenter&amp;author=X.+Huang&amp;volume=24&amp;publication_year=2018&amp;pages=3347-3358&amp;pmid=29879881&amp;doi=10.2174/1381612824666180607124038&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42-pharmaceuticals-15-00646">42. <span class="element-citation">Lionta E., Spyrou G., Vassilatis D.K., Cournia Z. Structure-based virtual screening for drug discovery: Principles, applications and recent advances. <span><span class="ref-journal">Curr. Top. Med. Chem. </span>2014;<span class="ref-vol">14</span>:1923–1938. doi: 10.2174/1568026614666140929124445.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4443793/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25262799" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2174%2F1568026614666140929124445" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Top.+Med.+Chem.&amp;title=Structure-based+virtual+screening+for+drug+discovery:+Principles,+applications+and+recent+advances&amp;author=E.+Lionta&amp;author=G.+Spyrou&amp;author=D.K.+Vassilatis&amp;author=Z.+Cournia&amp;volume=14&amp;publication_year=2014&amp;pages=1923-1938&amp;pmid=25262799&amp;doi=10.2174/1568026614666140929124445&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43-pharmaceuticals-15-00646">43. <span class="element-citation">Tyagi M., Begnini F., Poongavanam V., Doak B.C., Kihlberg J. Drug Syntheses Beyond the Rule of 5. <span><span class="ref-journal">Chemistry. </span>2020;<span class="ref-vol">26</span>:49–88. doi: 10.1002/chem.201902716.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31483909" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fchem.201902716" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chemistry&amp;title=Drug+Syntheses+Beyond+the+Rule+of+5&amp;author=M.+Tyagi&amp;author=F.+Begnini&amp;author=V.+Poongavanam&amp;author=B.C.+Doak&amp;author=J.+Kihlberg&amp;volume=26&amp;publication_year=2020&amp;pages=49-88&amp;pmid=31483909&amp;doi=10.1002/chem.201902716&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B44-pharmaceuticals-15-00646">44. <span class="element-citation">Allen W.J., Balius T.E., Mukherjee S., Brozell S.R., Moustakas D.T., Lang P.T., Case D.A., Kuntz I.D., Rizzo R.C. DOCK 6: Impact of new features and current docking performance. <span><span class="ref-journal">J. Comput. Chem. </span>2015;<span class="ref-vol">36</span>:1132–1156. doi: 10.1002/jcc.23905.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4469538/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25914306" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fjcc.23905" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Chem.&amp;title=DOCK+6:+Impact+of+new+features+and+current+docking+performance&amp;author=W.J.+Allen&amp;author=T.E.+Balius&amp;author=S.+Mukherjee&amp;author=S.R.+Brozell&amp;author=D.T.+Moustakas&amp;volume=36&amp;publication_year=2015&amp;pages=1132-1156&amp;pmid=25914306&amp;doi=10.1002/jcc.23905&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B45-pharmaceuticals-15-00646">45. <span class="element-citation">Morris G.M., Huey R., Olson A.J.  <span class="ref-journal">Current Protocols in Bioinformatics.</span> John Wiley &#x00026; Sons, Inc.; Hoboken, NJ, USA: 2008. Using AutoDock for ligand-receptor docking. Chapter 8, Unit 8.14. [<a href="https://pubmed.ncbi.nlm.nih.gov/19085980" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Current+Protocols+in+Bioinformatics&amp;author=G.M.+Morris&amp;author=R.+Huey&amp;author=A.J.+Olson&amp;publication_year=2008&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B46-pharmaceuticals-15-00646">46. <span class="element-citation">Pal S., Kumar V., Kundu B., Bhattacharya D., Preethy N., Reddy M.P., Talukdar A. Ligand-based Pharmacophore Modeling, Virtual Screening and Molecular Docking Studies for Discovery of Potential Topoisomerase I Inhibitors. <span><span class="ref-journal">Comput. Struct. Biotechnol. J. </span>2019;<span class="ref-vol">17</span>:291–310. doi: 10.1016/j.csbj.2019.02.006.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6396084/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30867893" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.csbj.2019.02.006" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Comput.+Struct.+Biotechnol.+J.&amp;title=Ligand-based+Pharmacophore+Modeling,+Virtual+Screening+and+Molecular+Docking+Studies+for+Discovery+of+Potential+Topoisomerase+I+Inhibitors&amp;author=S.+Pal&amp;author=V.+Kumar&amp;author=B.+Kundu&amp;author=D.+Bhattacharya&amp;author=N.+Preethy&amp;volume=17&amp;publication_year=2019&amp;pages=291-310&amp;pmid=30867893&amp;doi=10.1016/j.csbj.2019.02.006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B47-pharmaceuticals-15-00646">47. <span class="element-citation">Boz E., Stein M. Accurate Receptor-Ligand Binding Free Energies from Fast QM Conformational Chemical Space Sampling. <span><span class="ref-journal">Int. J. Mol. Sci. </span>2021;<span class="ref-vol">22</span>:3078.  doi: 10.3390/ijms22063078.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8002627/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33802920" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fijms22063078" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Mol.+Sci.&amp;title=Accurate+Receptor-Ligand+Binding+Free+Energies+from+Fast+QM+Conformational+Chemical+Space+Sampling&amp;author=E.+Boz&amp;author=M.+Stein&amp;volume=22&amp;publication_year=2021&amp;pages=3078&amp;pmid=33802920&amp;doi=10.3390/ijms22063078&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B48-pharmaceuticals-15-00646">48. <span class="element-citation">Sun Z., Huai Z., He Q., Liu Z. A General Picture of Cucurbit[8]uril Host-Guest Binding. <span><span class="ref-journal">J. Chem. Inf. Model. </span>2021;<span class="ref-vol">61</span>:6107–6134. doi: 10.1021/acs.jcim.1c01208.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34818004" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.jcim.1c01208" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Model.&amp;title=A+General+Picture+of+Cucurbit[8]uril+Host-Guest+Binding&amp;author=Z.+Sun&amp;author=Z.+Huai&amp;author=Q.+He&amp;author=Z.+Liu&amp;volume=61&amp;publication_year=2021&amp;pages=6107-6134&amp;pmid=34818004&amp;doi=10.1021/acs.jcim.1c01208&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B49-pharmaceuticals-15-00646">49. <span class="element-citation">Case D.A. Normal mode analysis of protein dynamics. <span><span class="ref-journal">Curr. Opin. Struct. Biol. </span>1994;<span class="ref-vol">4</span>:285–290. doi: 10.1016/S0959-440X(94)90321-2.</span> [<a href="//doi.org/10.1016%2FS0959-440X(94)90321-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Struct.+Biol.&amp;title=Normal+mode+analysis+of+protein+dynamics&amp;author=D.A.+Case&amp;volume=4&amp;publication_year=1994&amp;pages=285-290&amp;doi=10.1016/S0959-440X(94)90321-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B50-pharmaceuticals-15-00646">50. <span class="element-citation">Gallicchio E., Levy R.M. Recent theoretical and computational advances for modeling pro-tein-ligand binding affinities. <span><span class="ref-journal">Adv. Protein Chem. Struct Biol. </span>2011;<span class="ref-vol">85</span>:27–80.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3445424/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21920321" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv.+Protein+Chem.+Struct+Biol.&amp;title=Recent+theoretical+and+computational+advances+for+modeling+pro-tein-ligand+binding+affinities&amp;author=E.+Gallicchio&amp;author=R.M.+Levy&amp;volume=85&amp;publication_year=2011&amp;pages=27-80&amp;pmid=21920321&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B51-pharmaceuticals-15-00646">51. <span class="element-citation">Genheden S., Ryde U. The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities. <span><span class="ref-journal">Expert Opin. Drug Discov. </span>2015;<span class="ref-vol">10</span>:449–461. doi: 10.1517/17460441.2015.1032936.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4487606/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25835573" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1517%2F17460441.2015.1032936" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Opin.+Drug+Discov.&amp;title=The+MM/PBSA+and+MM/GBSA+methods+to+estimate+ligand-binding+affinities&amp;author=S.+Genheden&amp;author=U.+Ryde&amp;volume=10&amp;publication_year=2015&amp;pages=449-461&amp;pmid=25835573&amp;doi=10.1517/17460441.2015.1032936&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B52-pharmaceuticals-15-00646">52. <span class="element-citation">Qiu L., Yan Y., Sun Z., Song J., Zhang J.Z.H. Interaction entropy for computational alanine scanning in protein-protein binding. <span><span class="ref-journal">Wires Comput. Mol. Sci. </span>2018;<span class="ref-vol">8</span>:e1342.  doi: 10.1002/wcms.1342.</span> [<a href="//doi.org/10.1002%2Fwcms.1342" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Wires+Comput.+Mol.+Sci.&amp;title=Interaction+entropy+for+computational+alanine+scanning+in+protein-protein+binding&amp;author=L.+Qiu&amp;author=Y.+Yan&amp;author=Z.+Sun&amp;author=J.+Song&amp;author=J.Z.H.+Zhang&amp;volume=8&amp;publication_year=2018&amp;pages=e1342&amp;doi=10.1002/wcms.1342&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B53-pharmaceuticals-15-00646">53. <span class="element-citation">Whitesides G.M., Krishnamurthy V.M. Designing ligands to bind proteins. <span><span class="ref-journal">Q. Rev. Biophys. </span>2005;<span class="ref-vol">38</span>:385–395. doi: 10.1017/S0033583506004240.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2712650/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16817982" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1017%2FS0033583506004240" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Q.+Rev.+Biophys.&amp;title=Designing+ligands+to+bind+proteins&amp;author=G.M.+Whitesides&amp;author=V.M.+Krishnamurthy&amp;volume=38&amp;publication_year=2005&amp;pages=385-395&amp;pmid=16817982&amp;doi=10.1017/S0033583506004240&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B54-pharmaceuticals-15-00646">54. <span class="element-citation">Polishchuk P., Kutlushina A., Bashirova D., Mokshyna O., Madzhidov T. Virtual Screening Using Pharmacophore Models Retrieved from Molecular Dynamic Simulations. <span><span class="ref-journal">Int. J. Mol. Sci. </span>2019;<span class="ref-vol">20</span>:5834.  doi: 10.3390/ijms20235834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6929024/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31757043" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fijms20235834" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Mol.+Sci.&amp;title=Virtual+Screening+Using+Pharmacophore+Models+Retrieved+from+Molecular+Dynamic+Simulations&amp;author=P.+Polishchuk&amp;author=A.+Kutlushina&amp;author=D.+Bashirova&amp;author=O.+Mokshyna&amp;author=T.+Madzhidov&amp;volume=20&amp;publication_year=2019&amp;pages=5834&amp;doi=10.3390/ijms20235834&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B55-pharmaceuticals-15-00646">55. <span class="element-citation">Wieder M., Garon A., Perricone U., Boresch S., Seidel T., Almerico A.M., Langer T. Common Hits Approach: Combining Pharmacophore Modeling and Molecular Dynamics Simulations. <span><span class="ref-journal">J. Chem. Inf. Model. </span>2017;<span class="ref-vol">57</span>:365–385. doi: 10.1021/acs.jcim.6b00674.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28072524" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.jcim.6b00674" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Model.&amp;title=Common+Hits+Approach:+Combining+Pharmacophore+Modeling+and+Molecular+Dynamics+Simulations&amp;author=M.+Wieder&amp;author=A.+Garon&amp;author=U.+Perricone&amp;author=S.+Boresch&amp;author=T.+Seidel&amp;volume=57&amp;publication_year=2017&amp;pages=365-385&amp;pmid=28072524&amp;doi=10.1021/acs.jcim.6b00674&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B56-pharmaceuticals-15-00646">56. <span class="element-citation">Lombino J., Gulotta M.R., De Simone G., Mekni N., De Rosa M., Carbone D., Parrino B., Cascioferro S.M., Diana P., Padova A., et al. Dynamic-shared Pharmacophore Approach as Tool to Design New Allosteric PRC2 Inhibitors, Targeting EED Binding Pocket. <span><span class="ref-journal">Mol. Inform. </span>2021;<span class="ref-vol">40</span>:e2000148.  doi: 10.1002/minf.202000148.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32833314" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fminf.202000148" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Inform.&amp;title=Dynamic-shared+Pharmacophore+Approach+as+Tool+to+Design+New+Allosteric+PRC2+Inhibitors,+Targeting+EED+Binding+Pocket&amp;author=J.+Lombino&amp;author=M.R.+Gulotta&amp;author=G.+De+Simone&amp;author=N.+Mekni&amp;author=M.+De+Rosa&amp;volume=40&amp;publication_year=2021&amp;pages=e2000148&amp;pmid=32833314&amp;doi=10.1002/minf.202000148&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B57-pharmaceuticals-15-00646">57. <span class="element-citation">Caporuscio F., Tafi A. Pharmacophore modelling: A forty year old approach and its modern synergies. <span><span class="ref-journal">Curr. Med. Chem. </span>2011;<span class="ref-vol">18</span>:2543–2553. doi: 10.2174/092986711795933669.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21568893" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2174%2F092986711795933669" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Med.+Chem.&amp;title=Pharmacophore+modelling:+A+forty+year+old+approach+and+its+modern+synergies&amp;author=F.+Caporuscio&amp;author=A.+Tafi&amp;volume=18&amp;publication_year=2011&amp;pages=2543-2553&amp;pmid=21568893&amp;doi=10.2174/092986711795933669&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B58-pharmaceuticals-15-00646">58. <span class="element-citation">Prathipati P., Dixit A., Saxena A.K. Computer-aided drug design: Integration of structure-based and ligand-based approaches in drug design. <span><span class="ref-journal">Curr. Comput. Aided Drug Des. </span>2007;<span class="ref-vol">3</span>:133–148. doi: 10.2174/157340907780809516.</span> [<a href="//doi.org/10.2174%2F157340907780809516" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Comput.+Aided+Drug+Des.&amp;title=Computer-aided+drug+design:+Integration+of+structure-based+and+ligand-based+approaches+in+drug+design&amp;author=P.+Prathipati&amp;author=A.+Dixit&amp;author=A.K.+Saxena&amp;volume=3&amp;publication_year=2007&amp;pages=133-148&amp;doi=10.2174/157340907780809516&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B59-pharmaceuticals-15-00646">59. <span class="element-citation">Ferreira L.G., Dos Santos R.N., Oliva G., Andricopulo A.D. Molecular docking and structure-based drug design strategies. <span><span class="ref-journal">Molecules. </span>2015;<span class="ref-vol">20</span>:13384–13421. doi: 10.3390/molecules200713384.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6332083/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26205061" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fmolecules200713384" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Molecular+docking+and+structure-based+drug+design+strategies&amp;author=L.G.+Ferreira&amp;author=R.N.+Dos+Santos&amp;author=G.+Oliva&amp;author=A.D.+Andricopulo&amp;volume=20&amp;publication_year=2015&amp;pages=13384-13421&amp;pmid=26205061&amp;doi=10.3390/molecules200713384&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B60-pharmaceuticals-15-00646">60. <span class="element-citation">Wolber G., Langer T. LigandScout: 3-D Pharmacophores Derived from Protein-Bound Ligands and Their Use as Virtual Screening Filters. <span><span class="ref-journal">J. Chem. Inf. Model. </span>2005;<span class="ref-vol">45</span>:160–169. doi: 10.1021/ci049885e.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15667141" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fci049885e" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Model.&amp;title=LigandScout:+3-D+Pharmacophores+Derived+from+Protein-Bound+Ligands+and+Their+Use+as+Virtual+Screening+Filters&amp;author=G.+Wolber&amp;author=T.+Langer&amp;volume=45&amp;publication_year=2005&amp;pages=160-169&amp;pmid=15667141&amp;doi=10.1021/ci049885e&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B61-pharmaceuticals-15-00646">61. <span class="element-citation">Salam N.K., Nuti R., Sherman W. Novel method for generating structure-based pharmacophores using energetic analysis. <span><span class="ref-journal">J. Chem. Inf. Model. </span>2009;<span class="ref-vol">49</span>:2356–2368. doi: 10.1021/ci900212v.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19761201" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fci900212v" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Model.&amp;title=Novel+method+for+generating+structure-based+pharmacophores+using+energetic+analysis&amp;author=N.K.+Salam&amp;author=R.+Nuti&amp;author=W.+Sherman&amp;volume=49&amp;publication_year=2009&amp;pages=2356-2368&amp;pmid=19761201&amp;doi=10.1021/ci900212v&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B62-pharmaceuticals-15-00646">62. <span class="element-citation">Sutter J., Li J., Maynard A.J., Goupil A., Luu T., Nadassy K. New features that improve the pharmacophore tools from Accelrys. <span><span class="ref-journal">Curr. Comput. Aided Drug Des. </span>2011;<span class="ref-vol">7</span>:173–180. doi: 10.2174/157340911796504305.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21726193" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2174%2F157340911796504305" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Comput.+Aided+Drug+Des.&amp;title=New+features+that+improve+the+pharmacophore+tools+from+Accelrys&amp;author=J.+Sutter&amp;author=J.+Li&amp;author=A.J.+Maynard&amp;author=A.+Goupil&amp;author=T.+Luu&amp;volume=7&amp;publication_year=2011&amp;pages=173-180&amp;pmid=21726193&amp;doi=10.2174/157340911796504305&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B63-pharmaceuticals-15-00646">63. <span class="element-citation">Li H., Sutter J., Hoffmann R.  HypoGen: An Automated System for Generating Predictive 3D Pharmacophore Models. In: G&#x000fc;ner O.F., editor. <span class="ref-journal">Pharmacophore Perception, Development, and Use in Drug Design.</span> International University Line; La Jolla, CA, USA: 2000.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Pharmacophore+Perception,+Development,+and+Use+in+Drug+Design&amp;author=H.+Li&amp;author=J.+Sutter&amp;author=R.+Hoffmann&amp;publication_year=2000&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B64-pharmaceuticals-15-00646">64. <span class="element-citation">Dixon S.L., Smondyrev A.M., Knoll E.H., Rao S.N., Shaw D.E., Friesner R.A. PHASE: A new engine for pharmacophore perception, 3D QSAR model development, and 3D database screening: 1. Methodology and preliminary results. <span><span class="ref-journal">J. Comput. Aided Mol. Des. </span>2006;<span class="ref-vol">20</span>:647–671. doi: 10.1007/s10822-006-9087-6.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17124629" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs10822-006-9087-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Aided+Mol.+Des.&amp;title=PHASE:+A+new+engine+for+pharmacophore+perception,+3D+QSAR+model+development,+and+3D+database+screening:+1.+Methodology+and+preliminary+results&amp;author=S.L.+Dixon&amp;author=A.M.+Smondyrev&amp;author=E.H.+Knoll&amp;author=S.N.+Rao&amp;author=D.E.+Shaw&amp;volume=20&amp;publication_year=2006&amp;pages=647-671&amp;pmid=17124629&amp;doi=10.1007/s10822-006-9087-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B65-pharmaceuticals-15-00646">65. <span class="element-citation">Cramer R.D., Patterson D.E., Bunce J.D. Comparative molecular field analysis (CoMFA). 1. Effect of shape on binding of steroids to carrier proteins. <span><span class="ref-journal">J. Am. Chem. Soc. </span>1988;<span class="ref-vol">10</span>:5959–5967. doi: 10.1021/ja00226a005.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22148765" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fja00226a005" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Chem.+Soc.&amp;title=Comparative+molecular+field+analysis+(CoMFA).+1.+Effect+of+shape+on+binding+of+steroids+to+carrier+proteins&amp;author=R.D.+Cramer&amp;author=D.E.+Patterson&amp;author=J.D.+Bunce&amp;volume=10&amp;publication_year=1988&amp;pages=5959-5967&amp;doi=10.1021/ja00226a005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B66-pharmaceuticals-15-00646">66. <span class="element-citation">Klebe G., Abraham U., Mietzner T. Molecular similarity indices in a comparative analysis (CoMSIA) of drug molecules to correlate and predict their biological activity. <span><span class="ref-journal">J. Med. Chem. </span>1994;<span class="ref-vol">37</span>:4130–4146. doi: 10.1021/jm00050a010.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7990113" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fjm00050a010" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=Molecular+similarity+indices+in+a+comparative+analysis+(CoMSIA)+of+drug+molecules+to+correlate+and+predict+their+biological+activity&amp;author=G.+Klebe&amp;author=U.+Abraham&amp;author=T.+Mietzner&amp;volume=37&amp;publication_year=1994&amp;pages=4130-4146&amp;pmid=7990113&amp;doi=10.1021/jm00050a010&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B67-pharmaceuticals-15-00646">67. <span class="element-citation">Chen J., Lai L. Pocket v.2: Further Developments on Receptor-Based Pharmacophore Modeling. <span><span class="ref-journal">J. Chem. Inf. Model. </span>2006;<span class="ref-vol">46</span>:2684–2691. doi: 10.1021/ci600246s.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17125208" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fci600246s" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Model.&amp;title=Pocket+v.2:+Further+Developments+on+Receptor-Based+Pharmacophore+Modeling&amp;author=J.+Chen&amp;author=L.+Lai&amp;volume=46&amp;publication_year=2006&amp;pages=2684-2691&amp;pmid=17125208&amp;doi=10.1021/ci600246s&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B68-pharmaceuticals-15-00646">68. <span class="element-citation">Schneidman-Duhovny D., Dror O., Inbar Y., Nussinov R., Wolfson H.J. PharmaGist: A webserver for ligand-based pharmacophore detection. <span><span class="ref-journal">Nucleic Acids Res. </span>2008;<span class="ref-vol">36</span>:W223–W228. doi: 10.1093/nar/gkn187.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2447755/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18424800" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkn187" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=PharmaGist:+A+webserver+for+ligand-based+pharmacophore+detection&amp;author=D.+Schneidman-Duhovny&amp;author=O.+Dror&amp;author=Y.+Inbar&amp;author=R.+Nussinov&amp;author=H.J.+Wolfson&amp;volume=36&amp;publication_year=2008&amp;pages=W223-W228&amp;pmid=18424800&amp;doi=10.1093/nar/gkn187&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B69-pharmaceuticals-15-00646">69. <span class="element-citation">Ragno R. www.3d-qsar.com: A web portal that brings 3-D QSAR to all electronic devices-the Py-CoMFA web application as tool to build models from pre-aligned datasets. <span><span class="ref-journal">J. Comput. Aided Mol. Des. </span>2019;<span class="ref-vol">33</span>:855–864. doi: 10.1007/s10822-019-00231-x.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31595406" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs10822-019-00231-x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Aided+Mol.+Des.&amp;title=www.3d-qsar.com:+A+web+portal+that+brings+3-D+QSAR+to+all+electronic+devices-the+Py-CoMFA+web+application+as+tool+to+build+models+from+pre-aligned+datasets&amp;author=R.+Ragno&amp;volume=33&amp;publication_year=2019&amp;pages=855-864&amp;pmid=31595406&amp;doi=10.1007/s10822-019-00231-x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B70-pharmaceuticals-15-00646">70. <span class="element-citation">Reddy A.S., Pati S.P., Kumar P.P., Pradeep H.N., Sastry G.N. Virtual screening in drug discovery&#x02014;A computational perspective. <span><span class="ref-journal">Curr. Protein Pept. Sci. </span>2007;<span class="ref-vol">8</span>:329–351. doi: 10.2174/138920307781369427.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17696867" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2174%2F138920307781369427" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Protein+Pept.+Sci.&amp;title=Virtual+screening+in+drug+discovery&#x02014;A+computational+perspective&amp;author=A.S.+Reddy&amp;author=S.P.+Pati&amp;author=P.P.+Kumar&amp;author=H.N.+Pradeep&amp;author=G.N.+Sastry&amp;volume=8&amp;publication_year=2007&amp;pages=329-351&amp;pmid=17696867&amp;doi=10.2174/138920307781369427&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B71-pharmaceuticals-15-00646">71. <span class="element-citation">Kar S., Roy K. How far can virtual screening take us in drug discovery? <span><span class="ref-journal">Expert Opin. Drug Discov. </span>2013;<span class="ref-vol">8</span>:245–261. doi: 10.1517/17460441.2013.761204.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23330660" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1517%2F17460441.2013.761204" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Opin.+Drug+Discov.&amp;title=How+far+can+virtual+screening+take+us+in+drug+discovery?&amp;author=S.+Kar&amp;author=K.+Roy&amp;volume=8&amp;publication_year=2013&amp;pages=245-261&amp;pmid=23330660&amp;doi=10.1517/17460441.2013.761204&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B72-pharmaceuticals-15-00646">72. <span class="element-citation">Danishuddin M., Khan A.U. Structure based virtual screening to discover putative drug candidates: Necessary considerations and successful case studies. <span><span class="ref-journal">Methods. </span>2015;<span class="ref-vol">71</span>:135–145. doi: 10.1016/j.ymeth.2014.10.019.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25448480" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ymeth.2014.10.019" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Structure+based+virtual+screening+to+discover+putative+drug+candidates:+Necessary+considerations+and+successful+case+studies&amp;author=M.+Danishuddin&amp;author=A.U.+Khan&amp;volume=71&amp;publication_year=2015&amp;pages=135-145&amp;pmid=25448480&amp;doi=10.1016/j.ymeth.2014.10.019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B73-pharmaceuticals-15-00646">73. <span class="element-citation">Damm-Ganamet K.L., Arora N., Becart S., Edwards J.P., Lebsack A.D., McAllister H.M., Nelen M.I., Rao N.L., Westover L., Wiener J.J.M., et al. Accelerating Lead Identification by High Throughput Virtual Screening: Prospective Case Studies from the Pharmaceutical Industry. <span><span class="ref-journal">J. Chem. Inf. Model. </span>2019;<span class="ref-vol">59</span>:2046–2062. doi: 10.1021/acs.jcim.8b00941.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30817167" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.jcim.8b00941" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Chem.+Inf.+Model.&amp;title=Accelerating+Lead+Identification+by+High+Throughput+Virtual+Screening:+Prospective+Case+Studies+from+the+Pharmaceutical+Industry&amp;author=K.L.+Damm-Ganamet&amp;author=N.+Arora&amp;author=S.+Becart&amp;author=J.P.+Edwards&amp;author=A.D.+Lebsack&amp;volume=59&amp;publication_year=2019&amp;pages=2046-2062&amp;pmid=30817167&amp;doi=10.1021/acs.jcim.8b00941&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B74-pharmaceuticals-15-00646">74. <span class="element-citation">Temml V., Kutil Z. Structure-based molecular modeling in SAR analysis and lead optimization. <span><span class="ref-journal">Comput. Struct. Biotechnol. J. </span>2021;<span class="ref-vol">19</span>:1431–1444. doi: 10.1016/j.csbj.2021.02.018.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7979990/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33777339" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.csbj.2021.02.018" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Comput.+Struct.+Biotechnol.+J.&amp;title=Structure-based+molecular+modeling+in+SAR+analysis+and+lead+optimization&amp;author=V.+Temml&amp;author=Z.+Kutil&amp;volume=19&amp;publication_year=2021&amp;pages=1431-1444&amp;pmid=33777339&amp;doi=10.1016/j.csbj.2021.02.018&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B75-pharmaceuticals-15-00646">75. <span class="element-citation">Schuster D., Maurer E.M., Laggner C., Nashev L.G., Wilckens T., Langer T., Odermatt A. The Discovery of New 11&#x003b2;-Hydroxysteroid Dehydrogenase Type 1 Inhibitors by Common Feature Pharmacophore Modeling and Virtual Screening. <span><span class="ref-journal">J. Med. Chem. </span>2006;<span class="ref-vol">49</span>:3454–3466. doi: 10.1021/jm0600794.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16759088" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fjm0600794" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=The+Discovery+of+New+11&#x003b2;-Hydroxysteroid+Dehydrogenase+Type+1+Inhibitors+by+Common+Feature+Pharmacophore+Modeling+and+Virtual+Screening&amp;author=D.+Schuster&amp;author=E.M.+Maurer&amp;author=C.+Laggner&amp;author=L.G.+Nashev&amp;author=T.+Wilckens&amp;volume=49&amp;publication_year=2006&amp;pages=3454-3466&amp;pmid=16759088&amp;doi=10.1021/jm0600794&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B76-pharmaceuticals-15-00646">76. <span class="element-citation">Dotolo S., Cervellera C., Russo M., Russo G.L., Facchiano A. Virtual Screening of Natural Compounds as Potential PI3K-AKT1 Signaling Pathway Inhibitors and Experimental Validation. <span><span class="ref-journal">Molecules. </span>2021;<span class="ref-vol">26</span>:492.  doi: 10.3390/molecules26020492.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7831918/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33477701" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fmolecules26020492" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Virtual+Screening+of+Natural+Compounds+as+Potential+PI3K-AKT1+Signaling+Pathway+Inhibitors+and+Experimental+Validation&amp;author=S.+Dotolo&amp;author=C.+Cervellera&amp;author=M.+Russo&amp;author=G.L.+Russo&amp;author=A.+Facchiano&amp;volume=26&amp;publication_year=2021&amp;pages=492&amp;pmid=33477701&amp;doi=10.3390/molecules26020492&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B77-pharmaceuticals-15-00646">77. <span class="element-citation">Cervellera C., Russo M., Dotolo S., Facchiano A., Russo G.L. STL1, a New AKT Inhibitor, Synergizes with Flavonoid Quercetin in Enhancing Cell Death in A Chronic Lymphocytic Leukemia Cell Line. <span><span class="ref-journal">Molecules. </span>2021;<span class="ref-vol">26</span>:5810.  doi: 10.3390/molecules26195810.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8510450/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34641354" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fmolecules26195810" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=STL1,+a+New+AKT+Inhibitor,+Synergizes+with+Flavonoid+Quercetin+in+Enhancing+Cell+Death+in+A+Chronic+Lymphocytic+Leukemia+Cell+Line&amp;author=C.+Cervellera&amp;author=M.+Russo&amp;author=S.+Dotolo&amp;author=A.+Facchiano&amp;author=G.L.+Russo&amp;volume=26&amp;publication_year=2021&amp;pages=5810&amp;pmid=34641354&amp;doi=10.3390/molecules26195810&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B78-pharmaceuticals-15-00646">78. <span class="element-citation">Rahman N., Basharat Z., Yousuf M., Castaldo G., Rastrelli L., Khan H. Virtual Screening of Natural Products against Type II Transmembrane Serine Protease (TMPRSS2), the Priming Agent of Coronavirus 2 (SARS-CoV-2) <span><span class="ref-journal">Molecules. </span>2020;<span class="ref-vol">25</span>:2271.  doi: 10.3390/molecules25102271.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7287752/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32408547" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fmolecules25102271" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Virtual+Screening+of+Natural+Products+against+Type+II+Transmembrane+Serine+Protease+(TMPRSS2),+the+Priming+Agent+of+Coronavirus+2+(SARS-CoV-2)&amp;author=N.+Rahman&amp;author=Z.+Basharat&amp;author=M.+Yousuf&amp;author=G.+Castaldo&amp;author=L.+Rastrelli&amp;volume=25&amp;publication_year=2020&amp;pages=2271&amp;doi=10.3390/molecules25102271&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B79-pharmaceuticals-15-00646">79. <span class="element-citation">Rossi Sebastiano M., Ermondi G., Hadano S., Caron G. AI-based protein structure databases have the potential to accelerate rare diseases research: AlphaFoldDB and the case of IAHSP/Alsin. <span><span class="ref-journal">Drug Discov. Today. </span>2021;<span class="ref-vol">27</span>:1652–1660. doi: 10.1016/j.drudis.2021.12.018.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34958957" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.drudis.2021.12.018" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Discov.+Today&amp;title=AI-based+protein+structure+databases+have+the+potential+to+accelerate+rare+diseases+research:+AlphaFoldDB+and+the+case+of+IAHSP/Alsin&amp;author=M.+Rossi+Sebastiano&amp;author=G.+Ermondi&amp;author=S.+Hadano&amp;author=G.+Caron&amp;volume=27&amp;publication_year=2021&amp;pages=1652-1660&amp;pmid=34958957&amp;doi=10.1016/j.drudis.2021.12.018&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B80-pharmaceuticals-15-00646">80. <span class="element-citation">Song T., Zhang X., Ding M., Rodriguez-Paton A., Wang S., Wang G. DeepFusion: A Deep Learning Based Multi-Scale Feature Fusion Method for Predicting Drug-Target Interactions. <span><span class="ref-journal">Methods. </span>2022 doi: 10.1016/j.ymeth.2022.02.007.</span> in press. [<a href="https://pubmed.ncbi.nlm.nih.gov/35219861" ref="reftype=pubmed&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ymeth.2022.02.007" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=DeepFusion:+A+Deep+Learning+Based+Multi-Scale+Feature+Fusion+Method+for+Predicting+Drug-Target+Interactions&amp;author=T.+Song&amp;author=X.+Zhang&amp;author=M.+Ding&amp;author=A.+Rodriguez-Paton&amp;author=S.+Wang&amp;publication_year=2022&amp;doi=10.1016/j.ymeth.2022.02.007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=9145410&amp;issue-id=408698&amp;journal-id=2102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Pharmaceuticals</span> are provided here courtesy of <strong>Multidisciplinary Digital Publishing Institute  (MDPI)</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC9145410/pdf/pharmaceuticals-15-00646.pdf" class="int-view">PDF (6.7M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/9145410/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/9145410/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="9145410" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9145410%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="qVros8rPmAnuObK8oOQF30sTpWnf9Wzx9QQkggToB806vQooIQq2BSUX4mzN0d5y">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9145410%2F&amp;text=Drug%20Design%20by%20Pharmacophore%20and%20Virtual%20Screening%20Approach" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9145410%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9145410/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC9145410/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/35631472/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC9145410/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/35631472/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC9145410/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/9145410/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC9145410/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
